# ratify, bmeg-validator

A simple test of bmeg etl output


## setup

Get the data

The environmental variable DATA_DIR needs to point to
https://exastack-00.ohsu.edu/dashboard/project/containers/container/biostream/

How you get access is up to you.  You can download the data, or mount it.

For example to map it via [The Swift Virtual File System](https://github.com/ovh/svfs)

```
# after sourcing the openstack rc file.

$ sudo ~/Downloads/svfs-darwin-amd64 mount --device CCCDEV --mountpoint $(echo ~/CCC2)  --os-auth-url $OS_AUTH_URL --os-password $OS_PASSWORD --os-username $OS_USERNAME --os-tenant-name $OS_TENANT_NAME  --default-uid $(id -u)

```

## run the tests

```
# note: DATA_DIR defaults to 'biostream/protograph'
# note: SAMPLE_SIZE default to -1 (test all records)
$ DATA_DIR=<download/mapped dir> SAMPLE_SIZE=<number of records to test> py.test
```

## other

### protograph

The tests rely on the [yaml definitions of the graph](https://raw.githubusercontent.com/biostream/bmeg-etl/master/bmeg.protograph.yaml).
The test will download the yaml file to tests/ on first execution.
The tests will interrogate each file in the directory and compare it against the protograph definition.

Current checks include:
* vertex: ensure ['gid', 'data', 'label'] are ! empty
* edge: ensure ['to', 'fromLabel', 'from', 'gid', 'toLabel', 'label'] are ! empty.  Ensure that `data` obj has specified fields


### biostreams

The tests rely on protocol buffer definitions and python generated code.
**NOTE:** The proto files are manually committed to this repo.  Please update them.
Use `proto-generate.sh` to generate the python code

Current checks include:
* load json into proto object with ignore_unknown_fields=False



## example output

Each test will write a json formatted message to the standard python log output

### protograph

```
$ DATA_DIR=~/CCC2/biostream/protograph SAMPLE_SIZE=10 pytest  -s
============================= test session starts ==============================
platform darwin -- Python 2.7.13, pytest-3.0.6, py-1.4.34, pluggy-0.4.0
rootdir: /Users/walsbr/ratify, inifile:
plugins: flask-0.8.1, catchlog-1.2.2
collected 9 items

tests/test_protograph.py .F.FFFFFF

=================================== FAILURES ===================================
__________________________________ test_ccle ___________________________________

protograph = AttrDict({'Cohort': AttrDict({'vertexes': [{'filter': ['hasMember'], 'merge': ...'}, 'toLabel': 'Gene', 'label': 'variantIn'}], 'label': 'VariantAnnotation'})})

    def test_ccle(protograph):
        """ assert ccle data is ok """
        project_error_count = _validate_project(protograph, 'ccle')
>       assert project_error_count == 0
E       assert 29 == 0

tests/test_protograph.py:228: AssertionError
------------------------------ Captured log call -------------------------------
test_protograph.py          83 DEBUG    loading paths from /SWIFT/biostream/protograph/ccle
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'biosample::', u'data': {}, u'gid': u'(biosample::)--sampleOf->()', u'toLabel': u'Individual', u'label': u'sampleOf'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'biosample::', u'data': {}, u'gid': u'(biosample::)--sampleOf->()', u'toLabel': u'Individual', u'label': u'sampleOf'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'biosample::', u'data': {}, u'gid': u'(biosample::)--sampleOf->()', u'toLabel': u'Individual', u'label': u'sampleOf'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'biosample::', u'data': {}, u'gid': u'(biosample::)--sampleOf->()', u'toLabel': u'Individual', u'label': u'sampleOf'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'biosample::', u'data': {}, u'gid': u'(biosample::)--sampleOf->()', u'toLabel': u'Individual', u'label': u'sampleOf'})\nassert False\n +  where False = _exists('')"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Biosample.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.ResponseCurve.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.Variant.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.Variant.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.VariantAnnotation.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.VariantAnnotation.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.VariantAnnotation.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.ResponseCurve.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.ResponseCurve.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.Variant.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'', u'fromLabel': u'GeneExpression', u'from': u'geneExpression::GeneExpression', u'data': {}, u'gid': u'(geneExpression::GeneExpression)--expressionFor->()', u'toLabel': u'Biosample', u'label': u'expressionFor'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000252876.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000252876.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000252876.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000252876.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000267617.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000267617.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000267617.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000267617.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000215811.3', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000215811.3', u'level': 0.106453105807304}, u'gid': u'()--expressionLevel->(gene:ENSG00000215811.3)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000215811.3', u'level': 0.106453105807304})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000238020.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000238020.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000238020.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000238020.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000231620.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000231620.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000231620.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000231620.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000075624.9', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000075624.9', u'level': 864.669738769531}, u'gid': u'()--expressionLevel->(gene:ENSG00000075624.9)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000075624.9', u'level': 864.669738769531})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000248514.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000248514.1', u'level': 1.15239310264587}, u'gid': u'()--expressionLevel->(gene:ENSG00000248514.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000248514.1', u'level': 1.15239310264587})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000117676.9', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000117676.9', u'level': 13.0834789276123}, u'gid': u'()--expressionLevel->(gene:ENSG00000117676.9)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000117676.9', u'level': 13.0834789276123})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000265778.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000265778.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000265778.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000265778.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py         207 INFO     {"project": "ccle", "project_error_count": 29}
________________________ test_ccle_GeneExpression_Edge _________________________

protograph = AttrDict({'Cohort': AttrDict({'vertexes': [{'filter': ['hasMember'], 'merge': ...'}, 'toLabel': 'Gene', 'label': 'variantIn'}], 'label': 'VariantAnnotation'})})

    def test_ccle_GeneExpression_Edge(protograph):
        """ assert ccle data is ok """
        project_error_count = _validate_project(protograph, 'ccle', path_match=r'.*GeneExpression.Edge.json')  # noqa
>       assert project_error_count == 0
E       assert 19 == 0

tests/test_protograph.py:242: AssertionError
------------------------------ Captured log call -------------------------------
test_protograph.py          83 DEBUG    loading paths from /SWIFT/biostream/protograph/ccle
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'', u'fromLabel': u'GeneExpression', u'from': u'geneExpression::GeneExpression', u'data': {}, u'gid': u'(geneExpression::GeneExpression)--expressionFor->()', u'toLabel': u'Biosample', u'label': u'expressionFor'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000252876.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000252876.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000252876.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000252876.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000267617.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000267617.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000267617.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000267617.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000215811.3', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000215811.3', u'level': 0.106453105807304}, u'gid': u'()--expressionLevel->(gene:ENSG00000215811.3)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000215811.3', u'level': 0.106453105807304})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000238020.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000238020.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000238020.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000238020.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000231620.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000231620.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000231620.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000231620.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000075624.9', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000075624.9', u'level': 864.669738769531}, u'gid': u'()--expressionLevel->(gene:ENSG00000075624.9)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000075624.9', u'level': 864.669738769531})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000248514.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000248514.1', u'level': 1.15239310264587}, u'gid': u'()--expressionLevel->(gene:ENSG00000248514.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000248514.1', u'level': 1.15239310264587})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000117676.9', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000117676.9', u'level': 13.0834789276123}, u'gid': u'()--expressionLevel->(gene:ENSG00000117676.9)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000117676.9', u'level': 13.0834789276123})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'from' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'to': u'gene:ENSG00000265778.1', u'fromLabel': u'Biosample', u'from': u'', u'data': {u'sample': u'', u'gene': u'ENSG00000265778.1', u'level': 0.0}, u'gid': u'()--expressionLevel->(gene:ENSG00000265778.1)', u'toLabel': u'Gene', u'label': u'expressionLevel'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json", "error": "missing 'sample' in /SWIFT/biostream/protograph/ccle/ccle.GeneExpression.Edge.json AttrDict({u'sample': u'', u'gene': u'ENSG00000265778.1', u'level': 0.0})\nassert False\n +  where False = _exists('')"}
test_protograph.py         207 INFO     {"project": "ccle", "project_error_count": 19}
__________________________________ test_ctdd ___________________________________

protograph = AttrDict({'Cohort': AttrDict({'vertexes': [{'filter': ['hasMember'], 'merge': ...'}, 'toLabel': 'Gene', 'label': 'variantIn'}], 'label': 'VariantAnnotation'})})

    def test_ctdd(protograph):
        """ assert ctdd data is ok """
        project_error_count = _validate_project(protograph, 'ctdd')
>       assert project_error_count == 0
E       assert 11 == 0

tests/test_protograph.py:248: AssertionError
------------------------------ Captured log call -------------------------------
test_protograph.py          83 DEBUG    loading paths from /SWIFT/biostream/protograph/ctdd
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 0.6666666666666666, u'compound': u'pubchem:24771867'}, {u'ratio': 0.3333333333333333, u'compound': u'pubchem:24978538'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 0.2736}, {u'type': u'AUC', u'unit': u'uM', u'value': 7.9258}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/alisertib:navitoclax (2:1 mol/mol)', u'values': [{u'response': 0.2007, u'dose': 0.0015}, {u'response': -0.07458, u'dose': 0.003}, {u'response': -0.5947, u'dose': 0.0061}, {u'response': -0.01165, u'dose': 0.012}, {u'response': 0.03235, u'dose': 0.024}, {u'response': -0.08011, u'dose': 0.049}, {u'response': -0.6276, u'dose': 0.09699999999999999}, {u'response': -0.922, u'dose': 0.19}, {u'response': -1.266, u'dose': 0.39}, {u'response': -1.501, u'dose': 0.78}, {u'response': -1.454, u'dose': 1.6}, {u'response': -3.056, u'dose': 3.1}, {u'response': -3.397, u'dose': 6.2}, {u'response': -3.49, u'dose': 12.0}, {u'response': -3.62, u'dose': 25.0}, {u'response': -4.0280000000000005, u'dose': 50.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 0.5, u'compound': u'pubchem:24978538'}, {u'ratio': 0.5, u'compound': u'pubchem:216239'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 1.3319999999999999}, {u'type': u'AUC', u'unit': u'uM', u'value': 10.021}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/navitoclax:sorafenib (1:1 mol/mol)', u'values': [{u'response': 0.1961, u'dose': 0.0015}, {u'response': -0.0003637, u'dose': 0.003}, {u'response': 0.129, u'dose': 0.0061}, {u'response': 0.09207, u'dose': 0.012}, {u'response': -0.241, u'dose': 0.024}, {u'response': 0.1883, u'dose': 0.049}, {u'response': -0.020069999999999998, u'dose': 0.09699999999999999}, {u'response': 0.011290000000000001, u'dose': 0.19}, {u'response': -0.5448, u'dose': 0.39}, {u'response': -0.2828, u'dose': 0.78}, {u'response': -1.246, u'dose': 1.6}, {u'response': -1.827, u'dose': 3.1}, {u'response': -3.074, u'dose': 6.2}, {u'response': -2.6430000000000002, u'dose': 12.0}, {u'response': -4.47, u'dose': 25.0}, {u'response': -4.846, u'dose': 50.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 0.5, u'compound': u'pubchem:24978538'}, {u'ratio': 0.5, u'compound': u'pubchem:637858'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 0.9687}, {u'type': u'AUC', u'unit': u'uM', u'value': 10.331}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/navitoclax:piperlongumine (1:1 mol/mol)', u'values': [{u'response': -1.044, u'dose': 0.001}, {u'response': -0.061989999999999996, u'dose': 0.002}, {u'response': -0.0024460000000000003, u'dose': 0.0041}, {u'response': 0.08653, u'dose': 0.0081}, {u'response': -0.03174, u'dose': 0.016}, {u'response': -0.03232, u'dose': 0.032}, {u'response': 0.04825, u'dose': 0.065}, {u'response': -0.04052, u'dose': 0.13}, {u'response': -0.06175, u'dose': 0.26}, {u'response': -0.2726, u'dose': 0.52}, {u'response': -1.163, u'dose': 1.0}, {u'response': -1.6869999999999998, u'dose': 2.1}, {u'response': -3.18, u'dose': 4.2}, {u'response': -2.91, u'dose': 8.3}, {u'response': -4.0569999999999995, u'dose': 17.0}, {u'response': -3.9419999999999997, u'dose': 33.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 1.0, u'compound': u'pubchem:568763'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 4.47}, {u'type': u'AUC', u'unit': u'uM', u'value': 12.324}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/SMER-3', u'values': [{u'response': 0.04269, u'dose': 0.002}, {u'response': 0.031689999999999996, u'dose': 0.0041}, {u'response': 0.009276000000000001, u'dose': 0.0081}, {u'response': -0.008859, u'dose': 0.016}, {u'response': 0.002274, u'dose': 0.032}, {u'response': 0.1136, u'dose': 0.065}, {u'response': 0.044410000000000005, u'dose': 0.13}, {u'response': 0.009303, u'dose': 0.26}, {u'response': 0.09386, u'dose': 0.52}, {u'response': 0.06253, u'dose': 1.0}, {u'response': -0.08666, u'dose': 2.1}, {u'response': -0.9538, u'dose': 4.2}, {u'response': -2.8480000000000003, u'dose': 8.3}, {u'response': -2.8310000000000004, u'dose': 17.0}, {u'response': -0.3727, u'dose': 33.0}, {u'response': -1.217, u'dose': 66.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 1.0, u'compound': u'pubchem:119182'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 0.1058}, {u'type': u'AUC', u'unit': u'uM', u'value': 8.8058}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/clofarabine', u'values': [{u'response': 0.1249, u'dose': 0.002}, {u'response': 0.07028999999999999, u'dose': 0.0041}, {u'response': -0.7451, u'dose': 0.0081}, {u'response': -0.01061, u'dose': 0.016}, {u'response': -0.07529, u'dose': 0.032}, {u'response': -0.02767, u'dose': 0.065}, {u'response': -1.136, u'dose': 0.13}, {u'response': -1.787, u'dose': 0.26}, {u'response': -1.238, u'dose': 0.52}, {u'response': -1.3969999999999998, u'dose': 1.0}, {u'response': -1.357, u'dose': 2.1}, {u'response': -1.5930000000000002, u'dose': 4.2}, {u'response': -1.7819999999999998, u'dose': 8.3}, {u'response': -1.5759999999999998, u'dose': 17.0}, {u'response': -1.6369999999999998, u'dose': 33.0}, {u'response': -2.104, u'dose': 66.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 0.8, u'compound': u'pubchem:5311'}, {u'ratio': 0.2, u'compound': u'pubchem:24978538'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 1.128}, {u'type': u'AUC', u'unit': u'uM', u'value': 10.017}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/vorinostat:navitoclax (4:1 mol/mol)', u'values': [{u'response': -0.03573, u'dose': 0.001}, {u'response': -0.0399, u'dose': 0.002}, {u'response': -0.04513, u'dose': 0.0041}, {u'response': 0.031160000000000004, u'dose': 0.0081}, {u'response': -0.0524, u'dose': 0.016}, {u'response': -0.02814, u'dose': 0.032}, {u'response': -0.5619, u'dose': 0.065}, {u'response': -0.012209999999999999, u'dose': 0.13}, {u'response': -0.2488, u'dose': 0.26}, {u'response': -0.58, u'dose': 0.52}, {u'response': -0.7504, u'dose': 1.0}, {u'response': -1.485, u'dose': 2.1}, {u'response': -3.792, u'dose': 4.2}, {u'response': -4.018, u'dose': 8.3}, {u'response': -3.951, u'dose': 17.0}, {u'response': -4.104, u'dose': 33.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 1.0, u'compound': u'pubchem:49867926'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 0.8768}, {u'type': u'AUC', u'unit': u'uM', u'value': 13.941}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/XL765', u'values': [{u'response': 0.1919, u'dose': 0.002}, {u'response': 0.03658, u'dose': 0.0041}, {u'response': 0.07169, u'dose': 0.0081}, {u'response': 0.1118, u'dose': 0.016}, {u'response': 0.2408, u'dose': 0.032}, {u'response': -0.0533, u'dose': 0.065}, {u'response': 0.1392, u'dose': 0.13}, {u'response': 0.1842, u'dose': 0.26}, {u'response': 0.0193, u'dose': 0.52}, {u'response': -0.1525, u'dose': 1.0}, {u'response': -0.5424, u'dose': 2.1}, {u'response': 0.01891, u'dose': 4.2}, {u'response': -0.553, u'dose': 8.3}, {u'response': -0.5139, u'dose': 17.0}, {u'response': 0.00645, u'dose': 33.0}, {u'response': -0.2407, u'dose': 66.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 1.0, u'compound': u'pubchem:53301938'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 1.11}, {u'type': u'AUC', u'unit': u'uM', u'value': 16.409}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/ML258', u'values': [{u'response': 0.07468, u'dose': 0.002}, {u'response': -0.3932, u'dose': 0.0041}, {u'response': 0.1303, u'dose': 0.0081}, {u'response': 0.1859, u'dose': 0.016}, {u'response': 0.07153999999999999, u'dose': 0.032}, {u'response': 0.1737, u'dose': 0.065}, {u'response': -0.1329, u'dose': 0.13}, {u'response': 0.1472, u'dose': 0.26}, {u'response': 0.2489, u'dose': 0.52}, {u'response': 0.01346, u'dose': 1.0}, {u'response': 0.209, u'dose': 2.1}, {u'response': 0.2695, u'dose': 4.2}, {u'response': 0.1287, u'dose': 8.3}, {u'response': 0.2449, u'dose': 17.0}, {u'response': 0.2284, u'dose': 33.0}, {u'response': 0.2867, u'dose': 66.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 1.0, u'compound': u'pubchem:44241473'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 0.5053}, {u'type': u'AUC', u'unit': u'uM', u'value': 13.825}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/SR1001', u'values': [{u'response': 0.08559, u'dose': 0.002}, {u'response': 0.1308, u'dose': 0.0041}, {u'response': -0.4168, u'dose': 0.0081}, {u'response': 0.033710000000000004, u'dose': 0.016}, {u'response': 0.1404, u'dose': 0.032}, {u'response': 0.1775, u'dose': 0.065}, {u'response': 0.1135, u'dose': 0.13}, {u'response': 0.049530000000000005, u'dose': 0.26}, {u'response': -0.2004, u'dose': 0.52}, {u'response': -0.2995, u'dose': 1.0}, {u'response': -0.5728, u'dose': 2.1}, {u'response': -0.09107, u'dose': 4.2}, {u'response': -0.5243, u'dose': 8.3}, {u'response': -0.4473, u'dose': 17.0}, {u'response': 0.00916, u'dose': 33.0}, {u'response': -0.04193, u'dose': 66.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json", "error": "missing 'to' in /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Edge.json AttrDict({u'to': u'', u'fromLabel': u'Biosample', u'from': u'ccle:TUHR4TKB_KIDNEY', u'data': {u'responseType': u'ACTIVITY', u'compounds': [{u'ratio': 1.0, u'compound': u'pubchem:400769'}], u'_label': u'ResponseCurve', u'summary': [{u'type': u'EC50', u'unit': u'uM', u'value': 0.8052}, {u'type': u'AUC', u'unit': u'uM', u'value': 8.9512}], u'sample': u'ccle:TUHR4TKB_KIDNEY', u'gid': u'ccle-response:TUHR4TKB_KIDNEY/bardoxolone methyl', u'values': [{u'response': 0.2978, u'dose': 0.002}, {u'response': 0.0689, u'dose': 0.0041}, {u'response': 0.0071189999999999995, u'dose': 0.0081}, {u'response': -0.02263, u'dose': 0.016}, {u'response': -0.05786, u'dose': 0.032}, {u'response': 0.08782000000000001, u'dose': 0.065}, {u'response': 0.026619999999999998, u'dose': 0.13}, {u'response': -0.0394, u'dose': 0.26}, {u'response': -0.09788999999999999, u'dose': 0.52}, {u'response': -2.166, u'dose': 1.0}, {u'response': -3.537, u'dose': 2.1}, {u'response': -3.6460000000000004, u'dose': 4.2}, {u'response': -5.008, u'dose': 8.3}, {u'response': -5.471, u'dose': 17.0}, {u'response': -4.332, u'dose': 33.0}, {u'response': -5.206, u'dose': 66.0}]}, u'gid': u'(ccle:TUHR4TKB_KIDNEY)--responseTo->()', u'toLabel': u'Compound', u'label': u'responseTo'})\nassert False\n +  where False = _exists('')"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ctdd/ctdd.ResponseCurve.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         207 INFO     {"project": "ctdd", "project_error_count": 11}
_________________________________ test_ensembl _________________________________

protograph = AttrDict({'Cohort': AttrDict({'vertexes': [{'filter': ['hasMember'], 'merge': ...'}, 'toLabel': 'Gene', 'label': 'variantIn'}], 'label': 'VariantAnnotation'})})

    def test_ensembl(protograph):
        """ assert ensembl data is ok """
        project_error_count = _validate_project(protograph, 'ensembl')
>       assert project_error_count == 0
E       assert 3 == 0

tests/test_protograph.py:254: AssertionError
------------------------------ Captured log call -------------------------------
test_protograph.py          83 DEBUG    loading paths from /SWIFT/biostream/protograph/ensembl
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ensembl/ensembl.Exon.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ensembl/ensembl.Exon.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ensembl/ensembl.Exon.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ensembl/ensembl.Gene.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ensembl/ensembl.Gene.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ensembl/ensembl.Transcript.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ensembl/ensembl.Transcript.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/ensembl/ensembl.Transcript.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/ensembl/ensembl.Gene.Edge.json
test_protograph.py         207 INFO     {"project": "ensembl", "project_error_count": 3}
___________________________________ test_go ____________________________________

protograph = AttrDict({'Cohort': AttrDict({'vertexes': [{'filter': ['hasMember'], 'merge': ...'}, 'toLabel': 'Gene', 'label': 'variantIn'}], 'label': 'VariantAnnotation'})})

    def test_go(protograph):
        """ assert go data is ok """
        project_error_count = _validate_project(protograph, 'go')
>       assert project_error_count == 0
E       assert 1 == 0

tests/test_protograph.py:260: AssertionError
------------------------------ Captured log call -------------------------------
test_protograph.py          83 DEBUG    loading paths from /SWIFT/biostream/protograph/go
test_protograph.py          74 ERROR    {"path": "go", "error": ""}
test_protograph.py         207 INFO     {"project": "go", "project_error_count": 1}
___________________________________ test_mc3 ___________________________________

protograph = AttrDict({'Cohort': AttrDict({'vertexes': [{'filter': ['hasMember'], 'merge': ...'}, 'toLabel': 'Gene', 'label': 'variantIn'}], 'label': 'VariantAnnotation'})})

    def test_mc3(protograph):
        """ assert go data is ok """
        project_error_count = _validate_project(protograph, 'mc3')
>       assert project_error_count == 0
E       assert 2 == 0

tests/test_protograph.py:266: AssertionError
------------------------------ Captured log call -------------------------------
test_protograph.py          83 DEBUG    loading paths from /SWIFT/biostream/protograph/mc3
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/mc3/mc3.Variant.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/mc3/mc3.Variant.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/mc3/mc3.VariantAnnotation.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/mc3/mc3.VariantAnnotation.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/mc3/mc3.VariantAnnotation.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/mc3/mc3.Variant.Edge.json
test_protograph.py         207 INFO     {"project": "mc3", "project_error_count": 2}
__________________________________ test_tcga ___________________________________

protograph = AttrDict({'Cohort': AttrDict({'vertexes': [{'filter': ['hasMember'], 'merge': ...'}, 'toLabel': 'Gene', 'label': 'variantIn'}], 'label': 'VariantAnnotation'})})

    def test_tcga(protograph):
        """ assert go data is ok """
        project_error_count = _validate_project(protograph, 'tcga')
>       assert project_error_count == 0
E       assert 10 == 0

tests/test_protograph.py:272: AssertionError
------------------------------ Captured log call -------------------------------
test_protograph.py          83 DEBUG    loading paths from /SWIFT/biostream/protograph/tcga
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.Biosample.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.CNASegment.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.CNASegment.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.CNASegment.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.GeneExpression.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.GeneExpression.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.GeneExpression.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Individual.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.Individual.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json", "error": "missing 'data' in /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json AttrDict({u'gid': u'term:', u'data': {}, u'label': u'OntologyTerm'})\nassert False\n +  where False = _exists({})"}
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.Biosample.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.CNASegment.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.CNASegment.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.CNASegment.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.GeneExpression.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.GeneExpression.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.GeneExpression.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Individual.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.Individual.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.CNACallSet.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.CNACallSet.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.CNACallSet.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-BRCA.CNACallSet.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-BRCA.Individual.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.Individual.Edge.json
test_protograph.py         100 DEBUG    /SWIFT/biostream/protograph/tcga/TCGA-LUAD.CNACallSet.Vertex.json
test_protograph.py          74 ERROR    {"path": "/SWIFT/biostream/protograph/tcga/TCGA-LUAD.CNACallSet.Vertex.json", "error": "unsupported operand type(s) for +=: 'int' and 'ErrorCount'"}
test_protograph.py         207 INFO     {"project": "TCGA-LUAD", "project_error_count": 10}
===================== 7 failed, 2 passed in 17.12 seconds ======================

```



### biostreams

```
$ DATA_DIR=~/CCC2 SAMPLE_SIZE=10 pytest -s tests/test_biostream.py
============================= test session starts ==============================
platform darwin -- Python 2.7.13, pytest-3.0.6, py-1.4.34, pluggy-0.4.0
rootdir: /Users/walsbr/ratify, inifile:
plugins: flask-0.8.1, catchlog-1.2.2
collected 6 items

tests/test_biostream.py FF..FF

=================================== FAILURES ===================================
__________________________________ test_ccle ___________________________________

    def test_ccle():
        """ assert ccle data is ok """
        project_error_count = _validate_project('ccle')
>       assert project_error_count == 0
E       assert 30 == 0

tests/test_biostream.py:108: AssertionError
------------------------------ Captured log call -------------------------------
__init__.py                 51 DEBUG    loading paths from /SWIFT/biostream/biostream/ccle
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"astrocytoma\", \"gid\": \"ccle:1321N1_CENTRAL_NERVOUS_SYSTEM\", \"histology\": \"glioma\", \"tissue\": \"central_nervous_system\", \"alias\": \"1321N1\", \"source\": \"ECACC\", \"notes\": \"Identical lines: U-118 MG, U-138 MG and 1321N1 share high SNP identity\", \"type\": \"Sample\", \"sex\": \"M\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"NS\", \"gid\": \"ccle:143B_BONE\", \"histology\": \"osteosarcoma\", \"tissue\": \"bone\", \"alias\": \"143B\", \"source\": \"ATCC\", \"notes\": \"Identical lines: HTK-, HOS and 143B share high SNP identity and are very likely to be osteosarcoma\", \"type\": \"Sample\", \"sex\": \"F\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"NS\", \"gid\": \"ccle:22RV1_PROSTATE\", \"histology\": \"carcinoma\", \"tissue\": \"prostate\", \"alias\": \"22Rv1\", \"source\": \"ATCC\", \"notes\": \"\", \"type\": \"Sample\", \"sex\": \"M\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"adenocarcinoma\", \"gid\": \"ccle:2313287_STOMACH\", \"histology\": \"carcinoma\", \"tissue\": \"stomach\", \"alias\": \"23132/87\", \"source\": \"DSMZ\", \"notes\": \"\", \"type\": \"Sample\", \"sex\": \"M\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"transitional_cell_carcinoma\", \"gid\": \"ccle:253JBV_URINARY_TRACT\", \"histology\": \"carcinoma\", \"tissue\": \"urinary_tract\", \"alias\": \"253J-BV\", \"source\": \"KCLB\", \"notes\": \"Identical lines: 253J and 253J-BV share high SNP identity\", \"type\": \"Sample\", \"sex\": \"U\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"transitional_cell_carcinoma\", \"gid\": \"ccle:253J_URINARY_TRACT\", \"histology\": \"carcinoma\", \"tissue\": \"urinary_tract\", \"alias\": \"253J\", \"source\": \"KCLB\", \"notes\": \"Identical lines: 253J and 253J-BV share high SNP identity\", \"type\": \"Sample\", \"sex\": \"\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"astrocytoma_Grade_IV\", \"gid\": \"ccle:42MGBA_CENTRAL_NERVOUS_SYSTEM\", \"histology\": \"glioma\", \"tissue\": \"central_nervous_system\", \"alias\": \"42-MG-BA\", \"source\": \"DSMZ\", \"notes\": \"\", \"type\": \"Sample\", \"sex\": \"M\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"NS\", \"gid\": \"ccle:5637_URINARY_TRACT\", \"histology\": \"carcinoma\", \"tissue\": \"urinary_tract\", \"alias\": \"5637\", \"source\": \"ATCC\", \"notes\": \"\", \"type\": \"Sample\", \"sex\": \"M\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"NS\", \"gid\": \"ccle:59M_OVARY\", \"histology\": \"carcinoma\", \"tissue\": \"ovary\", \"alias\": \"59M\", \"source\": \"ECACC\", \"notes\": \"\", \"type\": \"Sample\", \"sex\": \"F\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.Biosample.json", "error": "pb parse e:Message type \"ga4gh.Biosample\" has no field named \"alias\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc06890>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc068d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06910>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06950>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06990>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc069d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a10>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a50>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06a90>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fc06ad0>] cls:Biosample {\"subtype\": \"transitional_cell_carcinoma\", \"gid\": \"ccle:639V_URINARY_TRACT\", \"histology\": \"carcinoma\", \"tissue\": \"urinary_tract\", \"alias\": \"639-V\", \"source\": \"DSMZ\", \"notes\": \"\", \"type\": \"Sample\", \"sex\": \"M\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:22RV1_PROSTATE\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:22RV1_PROSTATE\", \"name\": \"22RV1_PROSTATE\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:253J_URINARY_TRACT\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:253J_URINARY_TRACT\", \"name\": \"253J_URINARY_TRACT\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:253JBV_URINARY_TRACT\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:253JBV_URINARY_TRACT\", \"name\": \"253JBV_URINARY_TRACT\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:ACCMESO1_PLEURA\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:ACCMESO1_PLEURA\", \"name\": \"ACCMESO1_PLEURA\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:ALLSIL_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:ALLSIL_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"name\": \"ALLSIL_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:AML193_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:AML193_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"name\": \"AML193_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:AMO1_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:AMO1_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"name\": \"AMO1_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:BDCM_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:BDCM_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"name\": \"BDCM_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:BICR16_UPPER_AERODIGESTIVE_TRACT\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:BICR16_UPPER_AERODIGESTIVE_TRACT\", \"name\": \"BICR16_UPPER_AERODIGESTIVE_TRACT\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"ccle:biosample:BICR18_UPPER_AERODIGESTIVE_TRACT\", \"attributes\": {\"center\": null}, \"id\": \"ccle:variant:BICR18_UPPER_AERODIGESTIVE_TRACT\", \"name\": \"BICR18_UPPER_AERODIGESTIVE_TRACT\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 8.717773588, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 8.0, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -42.55801392}, {\"type\": \"ACTIVITY_AREA\", \"value\": 0.7124}], \"sample\": \"biosample:CCLE:1321N1_CENTRAL_NERVOUS_SYSTEM\", \"gid\": \"responseCurve:1321N1_CENTRAL_NERVOUS_SYSTEM:AEW541\", \"values\": [{\"response\": 8.67, \"dose\": 0.0025}, {\"response\": 11.0, \"dose\": 0.008}, {\"response\": 2.16, \"dose\": 0.025}, {\"response\": 0.27, \"dose\": 0.08}, {\"response\": -10.0, \"dose\": 0.25}, {\"response\": -13.0, \"dose\": 0.8}, {\"response\": -26.0, \"dose\": 2.53}, {\"response\": -43.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 8.165163646, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 2.329924107, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -71.58934021}, {\"type\": \"ACTIVITY_AREA\", \"value\": 1.6723}], \"sample\": \"biosample:CCLE:22RV1_PROSTATE\", \"gid\": \"responseCurve:22RV1_PROSTATE:AEW541\", \"values\": [{\"response\": 0.94, \"dose\": 0.0025}, {\"response\": 12.5, \"dose\": 0.008}, {\"response\": -14.0, \"dose\": 0.025}, {\"response\": 4.16, \"dose\": 0.08}, {\"response\": -25.0, \"dose\": 0.25}, {\"response\": -32.0, \"dose\": 0.8}, {\"response\": -52.0, \"dose\": 2.53}, {\"response\": -71.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 1.514508486, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 2.68212986, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -63.49137115}, {\"type\": \"ACTIVITY_AREA\", \"value\": 1.1852}], \"sample\": \"biosample:CCLE:42MGBA_CENTRAL_NERVOUS_SYSTEM\", \"gid\": \"responseCurve:42MGBA_CENTRAL_NERVOUS_SYSTEM:AEW541\", \"values\": [{\"response\": 8.91, \"dose\": 0.0025}, {\"response\": 8.39, \"dose\": 0.008}, {\"response\": -3.5, \"dose\": 0.025}, {\"response\": 12.4, \"dose\": 0.08}, {\"response\": -0.55, \"dose\": 0.25}, {\"response\": -6.2, \"dose\": 0.8}, {\"response\": -48.0, \"dose\": 2.53}, {\"response\": -63.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 8.006595166, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 5.002314091, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -62.35277557}, {\"type\": \"ACTIVITY_AREA\", \"value\": 0.9948}], \"sample\": \"biosample:CCLE:5637_URINARY_TRACT\", \"gid\": \"responseCurve:5637_URINARY_TRACT:AEW541\", \"values\": [{\"response\": 2.15, \"dose\": 0.0025}, {\"response\": 9.91, \"dose\": 0.008}, {\"response\": -3.5, \"dose\": 0.025}, {\"response\": 0.056, \"dose\": 0.08}, {\"response\": -2.1, \"dose\": 0.25}, {\"response\": -14.0, \"dose\": 0.8}, {\"response\": -30.0, \"dose\": 2.53}, {\"response\": -62.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 0.931195736, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 1.736181378, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -51.95980835}, {\"type\": \"ACTIVITY_AREA\", \"value\": 1.5436}], \"sample\": \"biosample:CCLE:639V_URINARY_TRACT\", \"gid\": \"responseCurve:639V_URINARY_TRACT:AEW541\", \"values\": [{\"response\": 11.8, \"dose\": 0.0025}, {\"response\": -7.3, \"dose\": 0.008}, {\"response\": -9.4, \"dose\": 0.025}, {\"response\": -15.0, \"dose\": 0.08}, {\"response\": -11.0, \"dose\": 0.25}, {\"response\": -21.0, \"dose\": 0.8}, {\"response\": -53.0, \"dose\": 2.53}, {\"response\": -50.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 8.700654505, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 4.260821819, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -73.33378601}, {\"type\": \"ACTIVITY_AREA\", \"value\": 1.7665}], \"sample\": \"biosample:CCLE:697_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE\", \"gid\": \"responseCurve:697_HAEMATOPOIETIC_AND_LYMPHOID_TISSUE:AEW541\", \"values\": [{\"response\": 18.4, \"dose\": 0.0025}, {\"response\": 4.65, \"dose\": 0.008}, {\"response\": 8.64, \"dose\": 0.025}, {\"response\": 19.3, \"dose\": 0.08}, {\"response\": -19.0, \"dose\": 0.25}, {\"response\": -20.0, \"dose\": 0.8}, {\"response\": -21.0, \"dose\": 2.53}, {\"response\": -78.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 0.306242794, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 8.0, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -39.68809509}, {\"type\": \"ACTIVITY_AREA\", \"value\": 1.4376}], \"sample\": \"biosample:CCLE:769P_KIDNEY\", \"gid\": \"responseCurve:769P_KIDNEY:AEW541\", \"values\": [{\"response\": 2.58, \"dose\": 0.0025}, {\"response\": -4.1, \"dose\": 0.008}, {\"response\": 6.18, \"dose\": 0.025}, {\"response\": -7.0, \"dose\": 0.08}, {\"response\": -13.0, \"dose\": 0.25}, {\"response\": -40.0, \"dose\": 0.8}, {\"response\": -28.0, \"dose\": 2.53}, {\"response\": -47.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 5.267667294, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 7.613147736, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -51.68631744}, {\"type\": \"ACTIVITY_AREA\", \"value\": 0.5954}], \"sample\": \"biosample:CCLE:786O_KIDNEY\", \"gid\": \"responseCurve:786O_KIDNEY:AEW541\", \"values\": [{\"response\": -1.2, \"dose\": 0.0025}, {\"response\": -0.53, \"dose\": 0.008}, {\"response\": 0.55, \"dose\": 0.025}, {\"response\": -4.1, \"dose\": 0.08}, {\"response\": 2.41, \"dose\": 0.25}, {\"response\": -11.0, \"dose\": 0.8}, {\"response\": -8.8, \"dose\": 2.53}, {\"response\": -52.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 8.520771485, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 4.95063591, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -66.22883606}, {\"type\": \"ACTIVITY_AREA\", \"value\": 1.2929}], \"sample\": \"biosample:CCLE:8305C_THYROID\", \"gid\": \"responseCurve:8305C_THYROID:AEW541\", \"values\": [{\"response\": 1.0, \"dose\": 0.0025}, {\"response\": -1.9, \"dose\": 0.008}, {\"response\": 8.83, \"dose\": 0.025}, {\"response\": 9.38, \"dose\": 0.08}, {\"response\": -18.0, \"dose\": 0.25}, {\"response\": -17.0, \"dose\": 0.8}, {\"response\": -28.0, \"dose\": 2.53}, {\"response\": -68.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ccle/ccle.ResponseCurve.json", "error": "pb parse e:Message type \"bmeg.ResponseCurve\" has no field named \"compound\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fb50350>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50390>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb503d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50410>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50450>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50490>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb504d0>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50510>, <google.protobuf.descriptor.FieldDescriptor object at 0x10fb50550>] cls:ResponseCurve {\"responseType\": \"ACTIVITY\", \"summary\": [{\"type\": \"EC50\", \"value\": 1.846689939, \"unit\": \"uM\"}, {\"type\": \"IC50\", \"value\": 8.0, \"unit\": \"uM\"}, {\"type\": \"AMAX\", \"value\": -22.26882553}, {\"type\": \"ACTIVITY_AREA\", \"value\": 0.3189}], \"sample\": \"biosample:CCLE:8505C_THYROID\", \"gid\": \"responseCurve:8505C_THYROID:AEW541\", \"values\": [{\"response\": -6.0, \"dose\": 0.0025}, {\"response\": -0.21, \"dose\": 0.008}, {\"response\": -2.4, \"dose\": 0.025}, {\"response\": 6.8, \"dose\": 0.08}, {\"response\": -6.3, \"dose\": 0.25}, {\"response\": 1.35, \"dose\": 0.8}, {\"response\": -19.0, \"dose\": 2.53}, {\"response\": -22.0, \"dose\": 8.0}], \"compound\": \"pubchem:11476171\"}\n"}
__________________________________ test_ctdd ___________________________________

    def test_ctdd():
        """ assert ctdd data is ok """
        project_error_count = _validate_project('ctdd')
>       assert project_error_count == 0
E       assert 1 == 0

tests/test_biostream.py:114: AssertionError
------------------------------ Captured log call -------------------------------
__init__.py                 51 DEBUG    loading paths from /SWIFT/biostream/biostream/ctdd
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/ctdd/ctdd.ResponseCurve.json", "error": "pb parse e:Failed to load JSON: Extra data: line 1 column 2 - line 2 column 1 (char 1 - 344). cls:ResponseCurve 005, \"dose\": 0.19}, {\"response\": -0.6924, \"dose\": 0.39}, {\"response\": -0.4604, \"dose\": 0.78}, {\"response\": -1.595, \"dose\": 1.6}, {\"response\": -5.257999999999999, \"dose\": 3.1}, {\"response\": -5.696000000000001, \"dose\": 6.2}, {\"response\": -5.681, \"dose\": 12.0}, {\"response\": -6.082000000000001, \"dose\": 25.0}, {\"response\": -6.187, \"dose\": 50.0}]}\n"}
___________________________________ test_mc3 ___________________________________

    def test_mc3():
        """ assert go data is ok """
        project_error_count = _validate_project('mc3')
>       assert project_error_count == 0
E       assert 10 == 0

tests/test_biostream.py:132: AssertionError
------------------------------ Captured log call -------------------------------
__init__.py                 51 DEBUG    loading paths from /SWIFT/biostream/biostream/mc3
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0003-01A-01D-1490-08\", \"attributes\": {\"center\": \"MUTECT\"}, \"id\": \"mc3:variant:TCGA-02-0003-01A-01D-1490-08:MUTECT\", \"name\": \"TCGA-02-0003-01A-01D-1490-08:MUTECT\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0003-01A-01D-1490-08\", \"attributes\": {\"center\": \"RADIA\"}, \"id\": \"mc3:variant:TCGA-02-0003-01A-01D-1490-08:RADIA\", \"name\": \"TCGA-02-0003-01A-01D-1490-08:RADIA\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0003-01A-01D-1490-08\", \"attributes\": {\"center\": \"SOMATICSNIPER\"}, \"id\": \"mc3:variant:TCGA-02-0003-01A-01D-1490-08:SOMATICSNIPER\", \"name\": \"TCGA-02-0003-01A-01D-1490-08:SOMATICSNIPER\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0003-01A-01D-1490-08\", \"attributes\": {\"center\": \"MUSE\"}, \"id\": \"mc3:variant:TCGA-02-0003-01A-01D-1490-08:MUSE\", \"name\": \"TCGA-02-0003-01A-01D-1490-08:MUSE\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0003-01A-01D-1490-08\", \"attributes\": {\"center\": \"VARSCANS\"}, \"id\": \"mc3:variant:TCGA-02-0003-01A-01D-1490-08:VARSCANS\", \"name\": \"TCGA-02-0003-01A-01D-1490-08:VARSCANS\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0033-01A-01D-1490-08\", \"attributes\": {\"center\": \"MUTECT\"}, \"id\": \"mc3:variant:TCGA-02-0033-01A-01D-1490-08:MUTECT\", \"name\": \"TCGA-02-0033-01A-01D-1490-08:MUTECT\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0033-01A-01D-1490-08\", \"attributes\": {\"center\": \"RADIA\"}, \"id\": \"mc3:variant:TCGA-02-0033-01A-01D-1490-08:RADIA\", \"name\": \"TCGA-02-0033-01A-01D-1490-08:RADIA\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0033-01A-01D-1490-08\", \"attributes\": {\"center\": \"SOMATICSNIPER\"}, \"id\": \"mc3:variant:TCGA-02-0033-01A-01D-1490-08:SOMATICSNIPER\", \"name\": \"TCGA-02-0033-01A-01D-1490-08:SOMATICSNIPER\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0033-01A-01D-1490-08\", \"attributes\": {\"center\": \"MUSE\"}, \"id\": \"mc3:variant:TCGA-02-0033-01A-01D-1490-08:MUSE\", \"name\": \"TCGA-02-0033-01A-01D-1490-08:MUSE\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/mc3/mc3.CallSet.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"center\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fc23cd0>] cls:CallSet {\"biosampleId\": \"mc3:biosample:TCGA-02-0033-01A-01D-1490-08\", \"attributes\": {\"center\": \"VARSCANS\"}, \"id\": \"mc3:variant:TCGA-02-0033-01A-01D-1490-08:VARSCANS\", \"name\": \"TCGA-02-0033-01A-01D-1490-08:VARSCANS\"}\n"}
__________________________________ test_tcga ___________________________________

    def test_tcga():
        """ assert go data is ok """
        project_error_count = _validate_project('tcga')
>       assert project_error_count == 0
E       assert 40 == 0

tests/test_biostream.py:138: AssertionError
------------------------------ Captured log call -------------------------------
__init__.py                 51 DEBUG    loading paths from /SWIFT/biostream/biostream/tcga
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"10\", \"oct_embedded\": \"false\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"BRCA\", \"project_code\": \"TCGA\", \"days_to_collection\": \"1407\", \"bcr_sample_uuid\": \"66CBC8DF-FF46-45F0-A2B4-2A5F4E6A3A2A\", \"bcr_patient_uuid\": \"4E509F73-99C7-48EB-A3A4-D34E3BFACE33\", \"days_to_index\": \"0\", \"sample_type\": \"Blood Derived Normal\", \"bcr_patient_barcode\": \"TCGA-OL-A5DA\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"gender\": \"FEMALE\", \"batch_number\": \"288.54.0\", \"patient_id\": \"A5DA\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"0967F358-424A-418E-866B-C9374397DC4C\", \"bcr_sample_barcode\": \"TCGA-OL-A5DA-10A\", \"tissue_source_site\": \"OL\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:4e509f73-99c7-48eb-a3a4-d34e3bface33\", \"id\": \"tcga:66cbc8df-ff46-45f0-a2b4-2a5f4e6a3a2a\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-OL-A5DA-10A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"11\", \"oct_embedded\": \"false\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"BRCA\", \"project_code\": \"TCGA\", \"year_of_dcc_upload\": \"2016\", \"bcr_sample_uuid\": \"065663ea-1f4d-4b94-9083-9199f696f09a\", \"bcr_patient_uuid\": \"3558dec7-eae3-4a41-a217-266c6a4535fb\", \"days_to_index\": \"0\", \"sample_type\": \"Solid Tissue Normal\", \"bcr_sample_barcode\": \"TCGA-E2-A1LI-11A\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"batch_number\": \"136.76.0\", \"patient_id\": \"A1LI\", \"days_to_collection\": \"2815\", \"file_uuid\": \"CF09155C-41DC-4A04-BE79-B64C934B0C46\", \"bcr_patient_barcode\": \"TCGA-E2-A1LI\", \"initial_weight\": \"190\", \"tissue_source_site\": \"E2\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:3558dec7-eae3-4a41-a217-266c6a4535fb\", \"id\": \"tcga:065663ea-1f4d-4b94-9083-9199f696f09a\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-E2-A1LI-11A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"oct_embedded\": \"false\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"BRCA\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-A8-A06Q.8FE77EF7-C83A-431F-B3BA-D93B5ECC70AF.pdf\", \"days_to_collection\": \"1184\", \"bcr_sample_uuid\": \"a03cd506-47aa-4333-a9a4-fa1a6815ac75\", \"bcr_patient_uuid\": \"281f70c5-876f-44b3-84fb-f2302f85e74c\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr_sample_barcode\": \"TCGA-A8-A06Q-01A\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"pathology_report_uuid\": \"8FE77EF7-C83A-431F-B3BA-D93B5ECC70AF\", \"batch_number\": \"56.90.0\", \"patient_id\": \"A06Q\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"C72BCFE0-CF56-494E-9283-47E7F114365B\", \"bcr_patient_barcode\": \"TCGA-A8-A06Q\", \"initial_weight\": \"50\", \"tissue_source_site\": \"A8\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:281f70c5-876f-44b3-84fb-f2302f85e74c\", \"id\": \"tcga:a03cd506-47aa-4333-a9a4-fa1a6815ac75\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-A8-A06Q-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"10\", \"oct_embedded\": \"false\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"BRCA\", \"project_code\": \"TCGA\", \"year_of_dcc_upload\": \"2016\", \"bcr_sample_uuid\": \"F0385728-4DBF-4D3E-9AB0-2217EC993F1A\", \"bcr_patient_uuid\": \"91B09291-5944-4691-A0D8-7D1CCC649B9B\", \"days_to_index\": \"0\", \"sample_type\": \"Blood Derived Normal\", \"bcr_sample_barcode\": \"TCGA-BH-A6R8-10A\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"gender\": \"FEMALE\", \"batch_number\": \"338.45.0\", \"patient_id\": \"A6R8\", \"days_to_collection\": \"260\", \"file_uuid\": \"73F36D22-67AB-4AAC-9F05-581A3B3EBDC0\", \"bcr_patient_barcode\": \"TCGA-BH-A6R8\", \"tissue_source_site\": \"BH\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:91b09291-5944-4691-a0d8-7d1ccc649b9b\", \"id\": \"tcga:f0385728-4dbf-4d3e-9ab0-2217ec993f1a\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-BH-A6R8-10A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"10\", \"oct_embedded\": \"false\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"BRCA\", \"project_code\": \"TCGA\", \"year_of_dcc_upload\": \"2016\", \"bcr_sample_uuid\": \"06a3d2e0-d0d4-4c80-9a74-bbba9997e928\", \"bcr_patient_uuid\": \"408cb583-6dc3-4698-8bd2-e284042bd5ef\", \"days_to_index\": \"0\", \"sample_type\": \"Blood Derived Normal\", \"bcr_sample_barcode\": \"TCGA-E9-A1RH-10A\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"gender\": \"FEMALE\", \"batch_number\": \"142.78.0\", \"patient_id\": \"A1RH\", \"days_to_collection\": \"32\", \"file_uuid\": \"E9BD4DC5-AC8F-4AA7-8F91-A88F1A3FE060\", \"bcr_patient_barcode\": \"TCGA-E9-A1RH\", \"tissue_source_site\": \"E9\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:408cb583-6dc3-4698-8bd2-e284042bd5ef\", \"id\": \"tcga:06a3d2e0-d0d4-4c80-9a74-bbba9997e928\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-E9-A1RH-10A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"bcr_sample_barcode\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"10\", \"day_of_dcc_upload\": \"31\", \"bcr_sample_uuid\": \"6ae136b4-1bd8-4567-bcdf-a3b96809674a\", \"is_ffpe\": \"NO\", \"bcr_patient_uuid\": \"3f834fa7-6d7b-4b85-98c0-5c55d55b6c95\", \"month_of_dcc_upload\": \"3\", \"days_to_index\": \"0\", \"batch_number\": \"56.90.0\", \"sample_type\": \"Blood Derived Normal\", \"patient_id\": \"A0IE\", \"days_to_collection\": \"5557\", \"file_uuid\": \"9586B897-C353-4371-A1BF-98F8700E91CD\", \"disease_code\": \"BRCA\", \"oct_embedded\": \"false\", \"year_of_dcc_upload\": \"2016\", \"tissue_source_site\": \"B6\", \"bcr_patient_barcode\": \"TCGA-B6-A0IE\", \"vial_number\": \"A\", \"project_code\": \"TCGA\", \"bcr\": \"Nationwide Children's Hospital\", \"bcr_sample_barcode\": \"TCGA-B6-A0IE-10A\"}, \"individualId\": \"tcga:3f834fa7-6d7b-4b85-98c0-5c55d55b6c95\", \"id\": \"tcga:6ae136b4-1bd8-4567-bcdf-a3b96809674a\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-B6-A0IE-10A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"10\", \"day_of_dcc_upload\": \"31\", \"oct_embedded\": \"false\", \"is_ffpe\": \"NO\", \"bcr_patient_uuid\": \"13b860c3-8048-4dc9-9eb0-7480bc660837\", \"project_code\": \"TCGA\", \"month_of_dcc_upload\": \"3\", \"days_to_index\": \"0\", \"batch_number\": \"56.90.0\", \"sample_type\": \"Blood Derived Normal\", \"vial_number\": \"A\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"08F725BB-0508-4A10-BB55-7BE92E887C4D\", \"disease_code\": \"BRCA\", \"bcr_sample_uuid\": \"a0e5f6e4-6f66-4fed-83cc-f63bbb9d5ba6\", \"bcr_sample_barcode\": \"TCGA-AN-A0FX-10A\", \"days_to_collection\": \"34\", \"patient_id\": \"A0FX\", \"tissue_source_site\": \"AN\", \"bcr\": \"Nationwide Children's Hospital\", \"bcr_patient_barcode\": \"TCGA-AN-A0FX\"}, \"individualId\": \"tcga:13b860c3-8048-4dc9-9eb0-7480bc660837\", \"id\": \"tcga:a0e5f6e4-6f66-4fed-83cc-f63bbb9d5ba6\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-AN-A0FX-10A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"10\", \"oct_embedded\": \"false\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"BRCA\", \"project_code\": \"TCGA\", \"days_to_collection\": \"32\", \"bcr_sample_uuid\": \"FD4C2168-F05E-4286-9ECD-647BF1BA9CE6\", \"bcr_patient_uuid\": \"00B11CA8-8540-4A3D-B602-EC754B00230B\", \"days_to_index\": \"0\", \"sample_type\": \"Blood Derived Normal\", \"bcr_patient_barcode\": \"TCGA-LL-A440\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"gender\": \"FEMALE\", \"batch_number\": \"255.56.0\", \"patient_id\": \"A440\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"BB20ADFF-D98D-4B47-8B96-01D5A69433CA\", \"bcr_sample_barcode\": \"TCGA-LL-A440-10A\", \"tissue_source_site\": \"LL\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:00b11ca8-8540-4a3d-b602-ec754b00230b\", \"id\": \"tcga:fd4c2168-f05e-4286-9ecd-647bf1ba9ce6\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-LL-A440-10A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"oct_embedded\": \"false\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"BRCA\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-A8-A09T.C616E97C-8352-4FED-9E83-558318216FB6.pdf\", \"days_to_collection\": \"394\", \"pathology_report_uuid\": \"C616E97C-8352-4FED-9E83-558318216FB6\", \"bcr_patient_uuid\": \"0adf59c6-581a-475d-a2f4-40aa40060b5b\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr_patient_barcode\": \"TCGA-A8-A09T\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"bcr_sample_uuid\": \"8df411bb-1d82-4470-84ff-e2d335698d42\", \"batch_number\": \"47.97.0\", \"patient_id\": \"A09T\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"B8F4D30B-CE2F-463B-8349-6FDABA79928D\", \"bcr_sample_barcode\": \"TCGA-A8-A09T-01A\", \"initial_weight\": \"100\", \"tissue_source_site\": \"A8\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:0adf59c6-581a-475d-a2f4-40aa40060b5b\", \"id\": \"tcga:8df411bb-1d82-4470-84ff-e2d335698d42\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-A8-A09T-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"oct_embedded\": \"true\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"BRCA\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-AR-A0TT.E9231B33-E1CE-4396-9F9E-03DBF049919A.pdf\", \"days_to_collection\": \"1567\", \"pathology_report_uuid\": \"E9231B33-E1CE-4396-9F9E-03DBF049919A\", \"bcr_patient_uuid\": \"13ca97a1-75bb-4a49-b8c3-181e5b2c9e86\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr_patient_barcode\": \"TCGA-AR-A0TT\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"bcr_sample_uuid\": \"8b5c92b7-52d5-4859-926f-f1264e9c1650\", \"batch_number\": \"72.90.0\", \"patient_id\": \"A0TT\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"FF25B74A-2480-4714-96B3-32F34E09979E\", \"bcr_sample_barcode\": \"TCGA-AR-A0TT-01A\", \"initial_weight\": \"310\", \"tissue_source_site\": \"AR\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:13ca97a1-75bb-4a49-b8c3-181e5b2c9e86\", \"id\": \"tcga:8b5c92b7-52d5-4859-926f-f1264e9c1650\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-AR-A0TT-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-19030\", \"axillary_staging_method\": \"Sentinel lymph node biopsy plus axillary dissection\", \"month_of_dcc_upload\": \"3\", \"er_status_by_ihc\": \"Positive\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"micromet_detection_by_ihc\": \"YES\", \"icd_o_3_site\": \"C50.9\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"vital_status\": \"Alive\", \"margin_status\": \"Negative\", \"project_code\": \"TCGA\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"491d5ebb-e9ab-4b17-acb8-630ff9fef0f2\", \"ajcc_staging_edition\": \"6th\", \"surgical_procedure_first\": \"Simple Mastectomy\", \"histologic_diagnosis\": \"Mixed Histology (please specify)\", \"menopause_status\": \"Pre (<6 months since LMP AND no prior bilateral ovariectomy AND not on estrogen replacement)\", \"pr_status_by_ihc\": \"Positive\", \"day_of_dcc_upload\": \"31\", \"initial_pathologic_dx_year\": \"2006\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"lymph_nodes_examined_count\": \"7\", \"her2_fish_status\": \"Negative\", \"icd_o_3_histology\": \"8522/3\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"er_status_ihc_Percent_Positive\": \"80-89%\", \"her2_ihc_score\": \"2+\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIA\", \"submitted_tumor_site\": \"Breast\", \"her2_status_by_ihc\": \"Equivocal\", \"form_completion_month\": \"8\", \"pr_status_ihc_percent_positive\": \"90-99%\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"YES\", \"lymph_nodes_examined\": \"YES\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"52\", \"batch_number\": \"142.78.0\", \"disease_code\": \"BRCA\", \"patient_id\": \"A24X\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"C36D7A39-09C9-4BE7-A420-F17C5A882277\", \"initial_pathologic_dx_days_to\": \"0\", \"bcr_patient_barcode\": \"TCGA-AR-A24X\", \"last_contact_days_to\": \"1465\", \"histologic_diagnosis_other\": \"Mixed invasive ductal and invasive lobular\", \"method_initial_path_dx\": \"Core needle biopsy\", \"lymph_nodes_examined_he_count\": \"0\", \"form_completion_day\": \"30\", \"tissue_source_site\": \"AR\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:491d5ebb-e9ab-4b17-acb8-630ff9fef0f2\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-AR-A24X\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-20346\", \"first_surgical_procedure_other\": \"Patey's Surgery\", \"tumor_status\": \"TUMOR FREE\", \"vital_status\": \"Alive\", \"month_of_dcc_upload\": \"3\", \"er_status_by_ihc\": \"Negative\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"YES\", \"icd_o_3_site\": \"C50.9\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"axillary_staging_method\": \"Axillary lymph node dissection alone\", \"metastatic_tumor_indicator\": \"NO\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"7e1673f8-5758-4963-8804-d5e39f06205b\", \"ajcc_staging_edition\": \"6th\", \"surgical_procedure_first\": \"Other\", \"histologic_diagnosis\": \"Infiltrating Ductal Carcinoma\", \"menopause_status\": \"Post (prior bilateral ovariectomy OR >12 mo since LMP with no prior hysterectomy)\", \"pr_status_by_ihc\": \"Negative\", \"day_of_dcc_upload\": \"31\", \"initial_pathologic_dx_year\": \"2010\", \"lymph_nodes_examined_he_count\": \"0\", \"lymph_nodes_examined_count\": \"5\", \"icd_o_3_histology\": \"8500/3\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"submitted_tumor_site\": \"Breast\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIA\", \"her2_status_by_ihc\": \"Negative\", \"form_completion_month\": \"3\", \"method_initial_path_dx_other\": \"Patey's Surgery\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"NO\", \"lymph_nodes_examined\": \"YES\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"55\", \"batch_number\": \"85.85.0\", \"disease_code\": \"BRCA\", \"patient_id\": \"A12V\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"0CE0D14A-8E68-4DBA-8C70-1FAF55337D3D\", \"bcr_patient_barcode\": \"TCGA-C8-A12V\", \"last_contact_days_to\": \"0\", \"method_initial_path_dx\": \"Other method, specify:\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"form_completion_day\": \"22\", \"tissue_source_site\": \"C8\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:7e1673f8-5758-4963-8804-d5e39f06205b\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-C8-A12V\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-30339\", \"first_surgical_procedure_other\": \"Right segmental mastectomy\", \"death_days_to\": \"1286\", \"tumor_status\": \"TUMOR FREE\", \"form_completion_day\": \"7\", \"month_of_dcc_upload\": \"3\", \"er_status_by_ihc\": \"Positive\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"icd_o_3_site\": \"C50.9\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"vital_status\": \"Dead\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T1\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"a1093598-d3a8-4ffe-83fc-bc7d1faff7e5\", \"ajcc_staging_edition\": \"5th\", \"histologic_diagnosis\": \"Infiltrating Ductal Carcinoma\", \"menopause_status\": \"Post (prior bilateral ovariectomy OR >12 mo since LMP with no prior hysterectomy)\", \"pr_status_by_ihc\": \"Positive\", \"initial_pathologic_dx_year\": \"1999\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"day_of_dcc_upload\": \"31\", \"icd_o_3_histology\": \"8500/3\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"submitted_tumor_site\": \"Breast\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage I\", \"her2_status_by_ihc\": \"Negative\", \"form_completion_month\": \"9\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"YES\", \"lymph_nodes_examined\": \"NO\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"83\", \"batch_number\": \"103.78.0\", \"disease_code\": \"BRCA\", \"patient_id\": \"A1EU\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"A4F0AC75-A279-4F45-8363-09088950501A\", \"bcr_patient_barcode\": \"TCGA-BH-A1EU\", \"method_initial_path_dx\": \"Fine needle aspiration biopsy\", \"tissue_source_site\": \"BH\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:a1093598-d3a8-4ffe-83fc-bc7d1faff7e5\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-BH-A1EU\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-21949\", \"her2_ihc_percent_positive\": \"<10%\", \"first_surgical_procedure_other\": \"Segmental mastectomy\", \"tumor_status\": \"TUMOR FREE\", \"vital_status\": \"Alive\", \"month_of_dcc_upload\": \"3\", \"age_at_initial_pathologic_diagnosis\": \"60\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"micromet_detection_by_ihc\": \"NO\", \"icd_o_3_site\": \"C50.9\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"axillary_staging_method\": \"Sentinel lymph node biopsy plus axillary dissection\", \"project_code\": \"TCGA\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"form_completion_year\": \"2010\", \"pr_positivity_ihc_intensity_score\": \"1+\", \"bcr_patient_uuid\": \"327a2395-85c8-42e1-9638-8e037a1723ed\", \"er_status_ihc_Percent_Positive\": \"90-99%\", \"surgical_procedure_first\": \"Other\", \"histologic_diagnosis\": \"Infiltrating Ductal Carcinoma\", \"menopause_status\": \"Post (prior bilateral ovariectomy OR >12 mo since LMP with no prior hysterectomy)\", \"pr_status_by_ihc\": \"Positive\", \"day_of_dcc_upload\": \"31\", \"initial_pathologic_dx_year\": \"2007\", \"lymph_nodes_examined_he_count\": \"12\", \"lymph_nodes_examined_count\": \"18\", \"icd_o_3_histology\": \"8500/3\", \"er_status_by_ihc\": \"Positive\", \"er_ihc_score\": \"3+\", \"her2_ihc_score\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIIC\", \"submitted_tumor_site\": \"Breast\", \"her2_status_by_ihc\": \"Negative\", \"form_completion_month\": \"11\", \"pr_status_ihc_percent_positive\": \"30-39%\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"YES\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"gender\": \"FEMALE\", \"margin_status\": \"Negative\", \"batch_number\": \"72.90.0\", \"disease_code\": \"BRCA\", \"patient_id\": \"A0BC\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"F9C6E7A3-F0B5-4C5C-8C36-A6B79658B067\", \"initial_pathologic_dx_days_to\": \"0\", \"bcr_patient_barcode\": \"TCGA-BH-A0BC\", \"last_contact_days_to\": \"974\", \"method_initial_path_dx\": \"Core needle biopsy\", \"ajcc_nodes_pathologic_pn\": \"N3\", \"form_completion_day\": \"4\", \"tissue_source_site\": \"BH\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:327a2395-85c8-42e1-9638-8e037a1723ed\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-BH-A0BC\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-22798\", \"pr_positivity_define_method\": \"IHC\", \"death_days_to\": \"362\", \"tumor_status\": \"WITH TUMOR\", \"form_completion_day\": \"2\", \"month_of_dcc_upload\": \"3\", \"er_status_by_ihc\": \"Indeterminate\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"micromet_detection_by_ihc\": \"NO\", \"icd_o_3_site\": \"C50.9\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"vital_status\": \"Dead\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T3\", \"axillary_staging_method\": \"No axillary staging\", \"form_completion_year\": \"2010\", \"bcr_patient_uuid\": \"44954053-0b93-4577-a436-42777e442300\", \"ajcc_staging_edition\": \"3rd\", \"histologic_diagnosis\": \"Infiltrating Ductal Carcinoma\", \"menopause_status\": \"Indeterminate (neither Pre or Postmenopausal)\", \"pr_status_by_ihc\": \"Positive\", \"initial_pathologic_dx_year\": \"1992\", \"ajcc_nodes_pathologic_pn\": \"NX\", \"day_of_dcc_upload\": \"31\", \"icd_o_3_histology\": \"8500/3\", \"submitted_tumor_site\": \"Breast\", \"initial_pathologic_dx_days_to\": \"0\", \"er_positivity_method\": \"IHC\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IV\", \"form_completion_month\": \"11\", \"method_initial_path_dx_other\": \"Biopsy, NOS\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"YES\", \"ajcc_metastasis_pathologic_pm\": \"M1\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"62\", \"batch_number\": \"56.90.0\", \"disease_code\": \"BRCA\", \"patient_id\": \"A0I9\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"55E98AA1-A2F9-4E7A-A4EF-0723107D34D7\", \"bcr_patient_barcode\": \"TCGA-B6-A0I9\", \"method_initial_path_dx\": \"Other method, specify:\", \"tissue_source_site\": \"B6\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:44954053-0b93-4577-a436-42777e442300\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-B6-A0I9\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-27287\", \"tumor_status\": \"TUMOR FREE\", \"form_completion_day\": \"26\", \"month_of_dcc_upload\": \"3\", \"er_status_by_ihc\": \"Positive\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"YES\", \"micromet_detection_by_ihc\": \"YES\", \"icd_o_3_site\": \"C50.9\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"vital_status\": \"Alive\", \"metastatic_tumor_indicator\": \"NO\", \"margin_status\": \"Negative\", \"file_uuid\": \"73D88F72-4C32-4DF8-9D15-777818D59E95\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T3\", \"axillary_staging_method\": \"Sentinel node biopsy alone\", \"er_positivity_scale_other\": \"Strong\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"cd0c96a4-6414-4efb-879b-7408b62ed5b6\", \"ajcc_staging_edition\": \"7th\", \"surgical_procedure_first\": \"Simple Mastectomy\", \"histologic_diagnosis\": \"Infiltrating Lobular Carcinoma\", \"menopause_status\": \"Post (prior bilateral ovariectomy OR >12 mo since LMP with no prior hysterectomy)\", \"pr_status_by_ihc\": \"Positive\", \"day_of_dcc_upload\": \"31\", \"initial_pathologic_dx_year\": \"2010\", \"ajcc_nodes_pathologic_pn\": \"N0 (i+)\", \"lymph_nodes_examined_count\": \"2\", \"her2_fish_status\": \"Negative\", \"icd_o_3_histology\": \"8520/3\", \"her2_cent17_ratio\": \"0.92\", \"er_status_ihc_Percent_Positive\": \"90-99%\", \"her2_ihc_score\": \"1+\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIB\", \"submitted_tumor_site\": \"Breast\", \"her2_status_by_ihc\": \"Negative\", \"form_completion_month\": \"5\", \"pr_positivity_scale_other\": \"Strong\", \"pr_status_ihc_percent_positive\": \"<10%\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"NO\", \"lymph_nodes_examined\": \"YES\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"74\", \"batch_number\": \"109.88.0\", \"disease_code\": \"BRCA\", \"patient_id\": \"A1FV\", \"year_of_dcc_upload\": \"2016\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"initial_pathologic_dx_days_to\": \"0\", \"lymph_nodes_examined_ihc_count\": \"1\", \"bcr_patient_barcode\": \"TCGA-A2-A1FV\", \"last_contact_days_to\": \"461\", \"method_initial_path_dx\": \"Core needle biopsy\", \"lymph_nodes_examined_he_count\": \"0\", \"tissue_source_site\": \"A2\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:cd0c96a4-6414-4efb-879b-7408b62ed5b6\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-A2-A1FV\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-22844\", \"tumor_status\": \"TUMOR FREE\", \"form_completion_day\": \"12\", \"month_of_dcc_upload\": \"3\", \"lymph_nodes_examined_ihc_count\": \"0\", \"history_neoadjuvant_treatment\": \"No\", \"her2_status_by_ihc\": \"Negative\", \"micromet_detection_by_ihc\": \"YES\", \"icd_o_3_site\": \"C50.9\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"vital_status\": \"Alive\", \"radiation_treatment_adjuvant\": \"YES\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"axillary_staging_method\": \"Sentinel node biopsy alone\", \"form_completion_year\": \"2015\", \"bcr_patient_uuid\": \"5C59028F-B8FA-4811-8314-BE3EAED5F364\", \"ajcc_staging_edition\": \"7th\", \"surgical_procedure_first\": \"Modified Radical Mastectomy\", \"histologic_diagnosis\": \"Infiltrating Ductal Carcinoma\", \"pr_status_by_ihc\": \"Negative\", \"day_of_dcc_upload\": \"31\", \"initial_pathologic_dx_year\": \"2012\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"lymph_nodes_examined_count\": \"8\", \"icd_o_3_histology\": \"8500/3\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"er_status_by_ihc\": \"Negative\", \"initial_pathologic_dx_days_to\": \"0\", \"pharmaceutical_tx_adjuvant\": \"YES\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage II\", \"submitted_tumor_site\": \"Breast\", \"form_completion_month\": \"3\", \"margin_status\": \"Negative\", \"history_other_malignancy\": \"No\", \"lymph_nodes_examined\": \"YES\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"62\", \"batch_number\": \"334.45.0\", \"disease_code\": \"BRCA\", \"patient_id\": \"A6SB\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"D0F636EE-B8B7-4AEC-9361-3D66C83332F8\", \"bcr_patient_barcode\": \"TCGA-EW-A6SB\", \"last_contact_days_to\": \"760\", \"method_initial_path_dx\": \"Core needle biopsy\", \"lymph_nodes_examined_he_count\": \"0\", \"tissue_source_site\": \"EW\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:5c59028f-b8fa-4811-8314-be3eaed5f364\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-EW-A6SB\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-22708\", \"tumor_status\": \"TUMOR FREE\", \"form_completion_day\": \"10\", \"month_of_dcc_upload\": \"3\", \"er_status_by_ihc\": \"Negative\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"YES\", \"icd_o_3_site\": \"C50.9\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"vital_status\": \"Alive\", \"margin_status\": \"Negative\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"axillary_staging_method\": \"Sentinel lymph node biopsy plus axillary dissection\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"3afa1e93-1df8-4e4c-aaa4-557463f4bb77\", \"surgical_procedure_first\": \"Lumpectomy\", \"histologic_diagnosis\": \"Infiltrating Ductal Carcinoma\", \"menopause_status\": \"Post (prior bilateral ovariectomy OR >12 mo since LMP with no prior hysterectomy)\", \"pr_status_by_ihc\": \"Negative\", \"day_of_dcc_upload\": \"31\", \"initial_pathologic_dx_year\": \"2009\", \"ajcc_nodes_pathologic_pn\": \"N0 (i-)\", \"lymph_nodes_examined_count\": \"1\", \"icd_o_3_histology\": \"8500/3\", \"submitted_tumor_site\": \"Breast\", \"her2_ihc_score\": \"1+\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIA\", \"her2_status_by_ihc\": \"Negative\", \"form_completion_month\": \"1\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"NO\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"62\", \"batch_number\": \"85.85.0\", \"disease_code\": \"BRCA\", \"patient_id\": \"A0DA\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"06D5DF5C-1E14-41F3-9012-D94CBCC409C8\", \"initial_pathologic_dx_days_to\": \"0\", \"bcr_patient_barcode\": \"TCGA-A7-A0DA\", \"last_contact_days_to\": \"373\", \"method_initial_path_dx\": \"Core needle biopsy\", \"lymph_nodes_examined_he_count\": \"0\", \"tissue_source_site\": \"A7\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:3afa1e93-1df8-4e4c-aaa4-557463f4bb77\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-A7-A0DA\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-17490\", \"first_surgical_procedure_other\": \"Wide local excision\", \"age_at_initial_pathologic_diagnosis\": \"47\", \"her2_fish_status\": \"Negative\", \"month_of_dcc_upload\": \"3\", \"er_status_by_ihc\": \"Positive\", \"history_neoadjuvant_treatment\": \"No\", \"her2_status_by_ihc\": \"Equivocal\", \"micromet_detection_by_ihc\": \"YES\", \"disease_code\": \"BRCA\", \"tumorSite\": \"Breast\", \"icd_10\": \"C50.9\", \"form_completion_day\": \"16\", \"file_uuid\": \"4AFA0D70-566B-444A-B446-57D8F9707702\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T1\", \"axillary_staging_method\": \"Sentinel lymph node biopsy plus axillary dissection\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"d1fd8ed2-be13-4272-b31e-df86911db3b1\", \"ajcc_staging_edition\": \"6th\", \"surgical_procedure_first\": \"Other\", \"histologic_diagnosis\": \"Infiltrating Ductal Carcinoma\", \"vital_status\": \"Alive\", \"pr_status_by_ihc\": \"Positive\", \"initial_pathologic_dx_year\": \"2005\", \"lymph_nodes_examined_he_count\": \"0\", \"tissue_source_site\": \"AR\", \"day_of_dcc_upload\": \"31\", \"icd_o_3_histology\": \"8500/3\", \"submitted_tumor_site\": \"Breast\", \"menopause_status\": \"Post (prior bilateral ovariectomy OR >12 mo since LMP with no prior hysterectomy)\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage I\", \"prospective_collection\": \"NO\", \"form_completion_month\": \"8\", \"pr_status_ihc_percent_positive\": \"10-19%\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"YES\", \"lymph_nodes_examined\": \"YES\", \"gender\": \"FEMALE\", \"margin_status\": \"Negative\", \"batch_number\": \"142.78.0\", \"icd_o_3_site\": \"C50.9\", \"patient_id\": \"A24P\", \"year_of_dcc_upload\": \"2016\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"her2_ihc_score\": \"2+\", \"bcr_patient_barcode\": \"TCGA-AR-A24P\", \"last_contact_days_to\": \"26\", \"lymph_nodes_examined_count\": \"3\", \"method_initial_path_dx\": \"Core needle biopsy\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"er_status_ihc_Percent_Positive\": \"10-19%\", \"bcr\": \"Nationwide Children's Hospital\", \"patient_withdrawal\": \"\\n            \\n        \"}, \"id\": \"tcga:d1fd8ed2-be13-4272-b31e-df86911db3b1\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-AR-A24P\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-BRCA.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-12642\", \"tumor_status\": \"TUMOR FREE\", \"her2_fish_status\": \"Positive\", \"month_of_dcc_upload\": \"3\", \"er_status_by_ihc\": \"Positive\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"micromet_detection_by_ihc\": \"YES\", \"disease_code\": \"BRCA\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C50.9\", \"form_completion_day\": \"9\", \"margin_status\": \"Negative\", \"patient_id\": \"A04X\", \"project_code\": \"TCGA\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"axillary_staging_method\": \"Sentinel node biopsy alone\", \"er_positivity_scale_other\": \"STRONG\", \"form_completion_year\": \"2010\", \"bcr_patient_uuid\": \"733a2a0f-b37a-4b81-b49e-3c0f30d1eb37\", \"ajcc_staging_edition\": \"6th\", \"surgical_procedure_first\": \"Simple Mastectomy\", \"histologic_diagnosis\": \"Infiltrating Ductal Carcinoma\", \"menopause_status\": \"Pre (<6 months since LMP AND no prior bilateral ovariectomy AND not on estrogen replacement)\", \"pr_status_by_ihc\": \"Positive\", \"day_of_dcc_upload\": \"31\", \"initial_pathologic_dx_year\": \"2006\", \"ajcc_nodes_pathologic_pn\": \"N0 (i-)\", \"lymph_nodes_examined_count\": \"3\", \"vital_status\": \"Alive\", \"icd_o_3_histology\": \"8500/3\", \"her2_cent17_ratio\": \"3.2\", \"submitted_tumor_site\": \"Breast\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIA\", \"her2_status_by_ihc\": \"Positive\", \"form_completion_month\": \"8\", \"pr_positivity_scale_other\": \"MODERATE\", \"pr_status_ihc_percent_positive\": \"50-59%\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"YES\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"34\", \"batch_number\": \"56.90.0\", \"icd_o_3_site\": \"C50.9\", \"tissue_source_site\": \"A2\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"7E1208C1-5C2E-4F60-9F1A-C38D6510A218\", \"her2_ihc_score\": \"3+\", \"lymph_nodes_examined_ihc_count\": \"0\", \"bcr_patient_barcode\": \"TCGA-A2-A04X\", \"last_contact_days_to\": \"1349\", \"method_initial_path_dx\": \"Core needle biopsy\", \"lymph_nodes_examined_he_count\": \"0\", \"er_status_ihc_Percent_Positive\": \"80-89%\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Breast\"}, \"id\": \"tcga:733a2a0f-b37a-4b81-b49e-3c0f30d1eb37\", \"datasetId\": \"TCGA-BRCA\", \"name\": \"TCGA-A2-A04X\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"LUAD\", \"intermediate_dimension\": \"1\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-97-7553.85eedad8-21be-4b10-910b-5b71802ae52f.pdf\", \"pathology_report_uuid\": \"85eedad8-21be-4b10-910b-5b71802ae52f\", \"bcr_patient_uuid\": \"37c8d73a-45ae-40fc-ba9a-721b755c1160\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr\": \"Nationwide Children's Hospital\", \"bcr_sample_barcode\": \"TCGA-97-7553-01A\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"shortest_dimension\": \"0.4\", \"vial_number\": \"A\", \"bcr_sample_uuid\": \"d23900e0-0335-429c-b9b6-16fa06740f92\", \"batch_number\": \"166.66.0\", \"patient_id\": \"7553\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"3BF2CD21-B7C4-4326-9BBD-CADB523A5C03\", \"bcr_patient_barcode\": \"TCGA-97-7553\", \"tissue_source_site\": \"97\", \"longest_dimension\": \"1.5\"}, \"individualId\": \"tcga:37c8d73a-45ae-40fc-ba9a-721b755c1160\", \"id\": \"tcga:d23900e0-0335-429c-b9b6-16fa06740f92\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-97-7553-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"LUAD\", \"intermediate_dimension\": \"0.4\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-55-8090.8f75f207-fe12-44ba-86b2-444ab5578d45.pdf\", \"bcr_sample_uuid\": \"53a420fd-6e31-46ee-8ac6-b383f4305cd2\", \"bcr_patient_uuid\": \"4d51ee44-e6f4-4bcb-be28-e9df54b39a8d\", \"longest_dimension\": \"1\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr_patient_barcode\": \"TCGA-55-8090\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"shortest_dimension\": \"0.2\", \"vial_number\": \"A\", \"pathology_report_uuid\": \"8f75f207-fe12-44ba-86b2-444ab5578d45\", \"gender\": \"MALE\", \"batch_number\": \"213.59.0\", \"patient_id\": \"8090\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"5E7B9F7D-9374-4617-AAB0-6E5873E899FA\", \"bcr_sample_barcode\": \"TCGA-55-8090-01A\", \"tissue_source_site\": \"55\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:4d51ee44-e6f4-4bcb-be28-e9df54b39a8d\", \"id\": \"tcga:53a420fd-6e31-46ee-8ac6-b383f4305cd2\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-55-8090-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"day_of_dcc_upload\": \"11\", \"bcr_sample_uuid\": \"72972ebc-3153-4803-a5b4-e03e00f277db\", \"is_ffpe\": \"NO\", \"bcr_patient_uuid\": \"fddc4d29-1ae8-451d-a13e-8ef6492be201\", \"project_code\": \"TCGA\", \"month_of_dcc_upload\": \"12\", \"batch_number\": \"34.10.0\", \"sample_type\": \"Primary Tumor\", \"vial_number\": \"A\", \"year_of_dcc_upload\": \"2015\", \"file_uuid\": \"062C75CA-5132-4AD2-BA97-0F712A93FB81\", \"disease_code\": \"LUAD\", \"bcr_sample_barcode\": \"TCGA-17-Z030-01A\", \"patient_id\": \"Z030\", \"tissue_source_site\": \"17\", \"bcr\": \"Washington University\", \"bcr_patient_barcode\": \"TCGA-17-Z030\"}, \"individualId\": \"tcga:fddc4d29-1ae8-451d-a13e-8ef6492be201\", \"id\": \"tcga:72972ebc-3153-4803-a5b4-e03e00f277db\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-17-Z030-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"LUAD\", \"intermediate_dimension\": \"1.2\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-69-7980.a4a32cf5-43b8-4387-9873-d90f7000b134.pdf\", \"bcr_sample_uuid\": \"e54ef0f0-38a5-4226-9a6a-c76aa12de589\", \"bcr_patient_uuid\": \"aee86a89-0377-4080-b16c-408bfbe78687\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr_patient_barcode\": \"TCGA-69-7980\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"shortest_dimension\": \"0.4\", \"vial_number\": \"A\", \"pathology_report_uuid\": \"a4a32cf5-43b8-4387-9873-d90f7000b134\", \"longest_dimension\": \"1.6\", \"batch_number\": \"204.61.0\", \"patient_id\": \"7980\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"E72BE069-7AF2-4432-A6F9-06F2FC873929\", \"bcr_sample_barcode\": \"TCGA-69-7980-01A\", \"gender\": \"FEMALE\", \"tissue_source_site\": \"69\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:aee86a89-0377-4080-b16c-408bfbe78687\", \"id\": \"tcga:e54ef0f0-38a5-4226-9a6a-c76aa12de589\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-69-7980-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"LUAD\", \"intermediate_dimension\": \"0.3\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-55-7907.233ef961-8050-47e5-bb53-260a7de4174d.pdf\", \"bcr_sample_uuid\": \"88c65791-1a9c-4121-955a-9bd5ed1b0bba\", \"bcr_patient_uuid\": \"bf755bc2-b7aa-4c3c-8b3c-58b089b5cab6\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr\": \"Nationwide Children's Hospital\", \"bcr_patient_barcode\": \"TCGA-55-7907\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"shortest_dimension\": \"0.1\", \"vial_number\": \"A\", \"pathology_report_uuid\": \"233ef961-8050-47e5-bb53-260a7de4174d\", \"gender\": \"MALE\", \"batch_number\": \"196.63.0\", \"patient_id\": \"7907\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"20C7AC42-5417-4FF8-8CA3-70B381570AFF\", \"bcr_sample_barcode\": \"TCGA-55-7907-01A\", \"tissue_source_site\": \"55\", \"longest_dimension\": \"0.3\"}, \"individualId\": \"tcga:bf755bc2-b7aa-4c3c-8b3c-58b089b5cab6\", \"id\": \"tcga:88c65791-1a9c-4121-955a-9bd5ed1b0bba\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-55-7907-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"LUAD\", \"intermediate_dimension\": \"0.7\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-49-4487.c67031fc-f601-4bd2-b8fd-429a0664cc65.pdf\", \"pathology_report_uuid\": \"c67031fc-f601-4bd2-b8fd-429a0664cc65\", \"bcr_patient_uuid\": \"bd4e9a9f-17f3-4d33-92fc-fcc45dd0e7b4\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr\": \"Nationwide Children's Hospital\", \"bcr_patient_barcode\": \"TCGA-49-4487\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"shortest_dimension\": \"0.3\", \"vial_number\": \"A\", \"bcr_sample_uuid\": \"3e88613e-9b97-4dbc-a052-b6dfe8c09956\", \"batch_number\": \"144.69.0\", \"patient_id\": \"4487\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"FB23ED63-9A5A-4A39-82EC-287300C025AB\", \"bcr_sample_barcode\": \"TCGA-49-4487-01A\", \"tissue_source_site\": \"49\", \"longest_dimension\": \"0.9\"}, \"individualId\": \"tcga:bd4e9a9f-17f3-4d33-92fc-fcc45dd0e7b4\", \"id\": \"tcga:3e88613e-9b97-4dbc-a052-b6dfe8c09956\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-49-4487-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"11\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"LUAD\", \"intermediate_dimension\": \"0.5\", \"project_code\": \"TCGA\", \"bcr_sample_uuid\": \"7bd574da-43aa-40f4-9b95-e50682aed48f\", \"bcr_patient_uuid\": \"199386c2-bb53-4fad-a1b6-59ab216a4a50\", \"days_to_index\": \"0\", \"sample_type\": \"Solid Tissue Normal\", \"bcr\": \"Nationwide Children's Hospital\", \"bcr_patient_barcode\": \"TCGA-55-1596\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"shortest_dimension\": \"0.2\", \"vial_number\": \"A\", \"batch_number\": \"37.76.0\", \"patient_id\": \"1596\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"043FA784-4D8A-4AEA-B65D-900666EF2207\", \"bcr_sample_barcode\": \"TCGA-55-1596-11A\", \"tissue_source_site\": \"55\", \"longest_dimension\": \"1\"}, \"individualId\": \"tcga:199386c2-bb53-4fad-a1b6-59ab216a4a50\", \"id\": \"tcga:7bd574da-43aa-40f4-9b95-e50682aed48f\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-55-1596-11A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"LUAD\", \"intermediate_dimension\": \"1\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-44-4112.9f214c90-f9a8-458c-8a53-ca8e8b09a2b2.pdf\", \"pathology_report_uuid\": \"9f214c90-f9a8-458c-8a53-ca8e8b09a2b2\", \"bcr_patient_uuid\": \"68b86559-38b2-41f2-b66e-c3c2b628b14d\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr_sample_barcode\": \"TCGA-44-4112-01A\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"shortest_dimension\": \"0.4\", \"vial_number\": \"A\", \"bcr_sample_uuid\": \"6c206676-e511-4281-91f5-bfe91b3279a4\", \"bcr_patient_barcode\": \"TCGA-44-4112\", \"batch_number\": \"52.73.0\", \"patient_id\": \"4112\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"DCEB95B6-51E5-4C03-A602-963727412846\", \"bcr\": \"Nationwide Children's Hospital\", \"initial_weight\": \"220\", \"tissue_source_site\": \"44\", \"longest_dimension\": \"2\"}, \"individualId\": \"tcga:68b86559-38b2-41f2-b66e-c3c2b628b14d\", \"id\": \"tcga:6c206676-e511-4281-91f5-bfe91b3279a4\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-44-4112-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"day_of_dcc_upload\": \"11\", \"bcr_sample_uuid\": \"815de84c-9fdf-4fcd-94f6-ae66eff41456\", \"is_ffpe\": \"NO\", \"bcr_patient_uuid\": \"392ca7f3-d52e-4aa7-a9ba-4c7d9aa6401c\", \"project_code\": \"TCGA\", \"batch_number\": \"34.10.0\", \"month_of_dcc_upload\": \"12\", \"sample_type\": \"Primary Tumor\", \"vial_number\": \"A\", \"year_of_dcc_upload\": \"2015\", \"file_uuid\": \"97A99889-11F4-439A-985B-3DAB2DBBB93A\", \"disease_code\": \"LUAD\", \"bcr_sample_barcode\": \"TCGA-17-Z056-01A\", \"patient_id\": \"Z056\", \"tissue_source_site\": \"17\", \"bcr\": \"Washington University\", \"bcr_patient_barcode\": \"TCGA-17-Z056\"}, \"individualId\": \"tcga:392ca7f3-d52e-4aa7-a9ba-4c7d9aa6401c\", \"id\": \"tcga:815de84c-9fdf-4fcd-94f6-ae66eff41456\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-17-Z056-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Biosample.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"sample_type_id\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Biosample {\"attributes\": {\"sample_type_id\": \"01\", \"oct_embedded\": \"true\", \"month_of_dcc_upload\": \"3\", \"disease_code\": \"LUAD\", \"project_code\": \"TCGA\", \"pathology_report_file_name\": \"TCGA-49-AAR4.8982CE18-FA5A-4882-AAC4-F2AC6ED60C49.pdf\", \"days_to_collection\": \"2842\", \"bcr_sample_uuid\": \"3BCE57D8-32D5-4F56-B0A4-544C3FEE778F\", \"bcr_patient_uuid\": \"765AD5A5-134B-4680-A6D4-1DD113DC47CC\", \"days_to_index\": \"0\", \"sample_type\": \"Primary Tumor\", \"bcr_patient_barcode\": \"TCGA-49-AAR4\", \"day_of_dcc_upload\": \"31\", \"is_ffpe\": \"NO\", \"vial_number\": \"A\", \"pathology_report_uuid\": \"8982CE18-FA5A-4882-AAC4-F2AC6ED60C49\", \"gender\": \"MALE\", \"batch_number\": \"423.35.0\", \"patient_id\": \"AAR4\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"B1C57D4E-D404-4B2B-872D-0389C9DD91DA\", \"bcr_sample_barcode\": \"TCGA-49-AAR4-01A\", \"initial_weight\": \"200\", \"tissue_source_site\": \"49\", \"bcr\": \"Nationwide Children's Hospital\"}, \"individualId\": \"tcga:765ad5a5-134b-4680-a6d4-1dd113dc47cc\", \"id\": \"tcga:3bce57d8-32d5-4f56-b0a4-544c3fee778f\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-49-AAR4-01A\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-26272\", \"tobacco_smoking_pack_years_smoked\": \"6\", \"death_days_to\": \"173\", \"residual_tumor\": \"R0\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"anatomic_organ_subdivision\": \"L-Lower\", \"disease_code\": \"LUAD\", \"tumorSite\": \"Lung\", \"icd_10\": \"C34.3\", \"form_completion_day\": \"28\", \"project_code\": \"TCGA\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"ff9def3d-17e5-4ef6-b74e-933f11ed6f00\", \"ajcc_staging_edition\": \"6th\", \"egfr_mutation_status\": \"NO\", \"performance_status_timing\": \"Preoperative\", \"tobacco_smoking_history_indicator\": \"4\", \"kras_gene_analysis_indicator\": \"NO\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"vital_status\": \"Dead\", \"location_lung_parenchyma\": \"Peripheral Lung\", \"initial_pathologic_dx_year\": \"1999\", \"ajcc_nodes_pathologic_pn\": \"N2\", \"day_of_dcc_upload\": \"31\", \"ecog_score\": \"1\", \"icd_o_3_histology\": \"8255/3\", \"submitted_tumor_site\": \"Lung\", \"initial_pathologic_dx_days_to\": \"0\", \"tobacco_smoking_year_stopped\": \"1998\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIIA\", \"eml4_alk_translocation_status\": \"NO\", \"form_completion_month\": \"9\", \"history_other_malignancy\": \"No\", \"tobacco_smoking_year_started\": \"1988\", \"retrospective_collection\": \"YES\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"71\", \"batch_number\": \"166.66.0\", \"icd_o_3_site\": \"C34.3\", \"patient_id\": \"7146\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"E29B5F35-128E-4630-BD0E-5EE71487B9C0\", \"bcr_patient_barcode\": \"TCGA-78-7146\", \"tissue_source_site\": \"78\", \"bcr\": \"Nationwide Children's Hospital\", \"patient_withdrawal\": \"\\n            \\n        \"}, \"id\": \"tcga:ff9def3d-17e5-4ef6-b74e-933f11ed6f00\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-78-7146\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-29989\", \"tumor_status\": \"TUMOR FREE\", \"form_completion_day\": \"3\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"YES\", \"anatomic_organ_subdivision\": \"R-Upper\", \"disease_code\": \"LUAD\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C34.1\", \"vital_status\": \"Alive\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"673493f6-975c-49e8-934c-001e9a0fff90\", \"ajcc_staging_edition\": \"6th\", \"egfr_mutation_status\": \"NO\", \"icd_o_3_histology\": \"8140/3\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"initial_pathologic_dx_year\": \"2009\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"day_of_dcc_upload\": \"31\", \"kras_gene_analysis_indicator\": \"NO\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"submitted_tumor_site\": \"Lung\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IB\", \"eml4_alk_translocation_status\": \"NO\", \"form_completion_month\": \"3\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"NO\", \"tobacco_smoking_history_indicator\": \"1\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"82\", \"batch_number\": \"37.76.0\", \"icd_o_3_site\": \"C34.1\", \"patient_id\": \"3772\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"E3F790AC-CFB5-4DDC-8FAB-7AD23A35064D\", \"bcr_patient_barcode\": \"TCGA-67-3772\", \"last_contact_days_to\": \"573\", \"tissue_source_site\": \"67\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Lung\"}, \"id\": \"tcga:673493f6-975c-49e8-934c-001e9a0fff90\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-67-3772\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-20219\", \"tobacco_smoking_pack_years_smoked\": \"25\", \"pulmonary_function_test_indicator\": \"YES\", \"residual_tumor\": \"R0\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"anatomic_organ_subdivision\": \"R-Upper\", \"disease_code\": \"LUAD\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C34.1\", \"form_completion_day\": \"3\", \"file_uuid\": \"6447EDCB-2FC5-4374-B69D-0680A241AC81\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"41c4c704-825e-49f5-9d54-dbd7d30725e0\", \"ajcc_staging_edition\": \"6th\", \"fev1_percent_ref_prebroncholiator\": \"77\", \"performance_status_timing\": \"Post-Adjuvant Therapy\", \"carbon_monoxide_diffusion_dlco\": \"73\", \"vital_status\": \"Alive\", \"location_lung_parenchyma\": \"Central Lung\", \"tumor_status\": \"TUMOR FREE\", \"initial_pathologic_dx_year\": \"2007\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"day_of_dcc_upload\": \"31\", \"ecog_score\": \"0\", \"icd_o_3_histology\": \"8255/3\", \"submitted_tumor_site\": \"Lung\", \"initial_pathologic_dx_days_to\": \"0\", \"tobacco_smoking_year_stopped\": \"2007\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IB\", \"form_completion_month\": \"5\", \"history_other_malignancy\": \"No\", \"tobacco_smoking_year_started\": \"1982\", \"retrospective_collection\": \"YES\", \"tobacco_smoking_history_indicator\": \"4\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"55\", \"fev1_fvc_ratio_prebroncholiator\": \"61\", \"batch_number\": \"84.72.0\", \"icd_o_3_site\": \"C34.1\", \"patient_id\": \"5781\", \"year_of_dcc_upload\": \"2016\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"bcr_patient_barcode\": \"TCGA-64-5781\", \"last_contact_days_to\": \"1202\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"tissue_source_site\": \"64\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Lung\"}, \"id\": \"tcga:41c4c704-825e-49f5-9d54-dbd7d30725e0\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-64-5781\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-24725\", \"carbon_monoxide_diffusion_dlco\": \"90\", \"eml4_alk_analysis_type\": \"FISH\", \"tobacco_smoking_pack_years_smoked\": \"40\", \"pharmaceutical_therapy_adjuvant\": \"YES\", \"death_days_to\": \"164\", \"pulmonary_function_test_indicator\": \"YES\", \"vital_status\": \"Dead\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"YES\", \"anatomic_organ_subdivision\": \"R-Upper\", \"egfr_mutation_status\": \"YES\", \"disease_code\": \"LUAD\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C34.1\", \"form_completion_day\": \"13\", \"radiation_treatment_adjuvant\": \"NO\", \"file_uuid\": \"3CC3F939-7924-4A03-8489-E95F433F6955\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T2b\", \"form_completion_year\": \"2012\", \"bcr_patient_uuid\": \"8c695c24-e7f9-449a-9258-a1bc8a33bf9a\", \"ajcc_staging_edition\": \"7th\", \"fev1_percent_ref_prebroncholiator\": \"91\", \"performance_status_timing\": \"Preoperative\", \"icd_o_3_histology\": \"8260/3\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"initial_pathologic_dx_year\": \"2011\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"day_of_dcc_upload\": \"31\", \"ecog_score\": \"0\", \"kras_gene_analysis_indicator\": \"YES\", \"eml4_alk_translocation_status\": \"YES\", \"submitted_tumor_site\": \"Lung\", \"fev1_percent_ref_postbroncholiator\": \"99\", \"initial_pathologic_dx_days_to\": \"0\", \"tobacco_smoking_year_stopped\": \"1985\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIA\", \"kras_mutation_found\": \"NO\", \"form_completion_month\": \"6\", \"history_other_malignancy\": \"No\", \"tobacco_smoking_year_started\": \"1965\", \"fev1_fvc_ratio_postbroncholiator\": \"86\", \"retrospective_collection\": \"NO\", \"tobacco_smoking_history_indicator\": \"3\", \"gender\": \"MALE\", \"age_at_initial_pathologic_diagnosis\": \"67\", \"fev1_fvc_ratio_prebroncholiator\": \"81\", \"batch_number\": \"222.59.0\", \"icd_o_3_site\": \"C34.1\", \"patient_id\": \"8174\", \"year_of_dcc_upload\": \"2016\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"bcr_patient_barcode\": \"TCGA-97-8174\", \"tissue_source_site\": \"97\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Lung\"}, \"id\": \"tcga:8c695c24-e7f9-449a-9258-a1bc8a33bf9a\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-97-8174\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-21565\", \"tobacco_smoking_pack_years_smoked\": \"30\", \"pulmonary_function_test_indicator\": \"YES\", \"vital_status\": \"Alive\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"YES\", \"anatomic_organ_subdivision\": \"R-Upper\", \"icd_o_3_site\": \"C34.1\", \"tumorSite\": \"Lung\", \"icd_10\": \"C34.1\", \"form_completion_day\": \"5\", \"project_code\": \"TCGA\", \"carbon_monoxide_diffusion_dlco\": \"44\", \"ajcc_tumor_pathologic_pt\": \"T2b\", \"form_completion_year\": \"2013\", \"bcr_patient_uuid\": \"EA0B9461-0D4D-4ADE-A255-1B03DD85E30E\", \"ajcc_staging_edition\": \"7th\", \"egfr_mutation_status\": \"NO\", \"kras_gene_analysis_indicator\": \"NO\", \"tobacco_smoking_history_indicator\": \"4\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"initial_pathologic_dx_year\": \"2012\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"day_of_dcc_upload\": \"31\", \"icd_o_3_histology\": \"8140/3\", \"submitted_tumor_site\": \"Lung\", \"fev1_percent_ref_postbroncholiator\": \"69\", \"initial_pathologic_dx_days_to\": \"0\", \"tobacco_smoking_year_stopped\": \"2001\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIA\", \"form_completion_month\": \"2\", \"history_other_malignancy\": \"Yes, History of Synchronous/Bilateral Malignancy\", \"tobacco_smoking_year_started\": \"1976\", \"retrospective_collection\": \"NO\", \"ajcc_metastasis_pathologic_pm\": \"MX\", \"gender\": \"MALE\", \"age_at_initial_pathologic_diagnosis\": \"59\", \"batch_number\": \"258.58.0\", \"disease_code\": \"LUAD\", \"patient_id\": \"A4BC\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"E82437A1-5FFE-40DF-9A42-061E0A8F7883\", \"bcr_patient_barcode\": \"TCGA-91-A4BC\", \"last_contact_days_to\": \"44\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"tissue_source_site\": \"91\", \"bcr\": \"Nationwide Children's Hospital\", \"patient_withdrawal\": \"\\n            \\n        \"}, \"id\": \"tcga:ea0b9461-0d4d-4ade-a255-1b03dd85e30e\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-91-A4BC\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-19216\", \"carbon_monoxide_diffusion_dlco\": \"97\", \"tobacco_smoking_pack_years_smoked\": \"50\", \"tumor_status\": \"TUMOR FREE\", \"form_completion_day\": \"20\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"anatomic_organ_subdivision\": \"R-Upper\", \"icd_o_3_site\": \"C34.1\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C34.1\", \"vital_status\": \"Alive\", \"egfr_mutation_status\": \"NO\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"0b31dbdf-0623-48e4-a0ec-017650dceda7\", \"ajcc_staging_edition\": \"7th\", \"fev1_percent_ref_prebroncholiator\": \"108\", \"icd_o_3_histology\": \"8140/3\", \"file_uuid\": \"F16503A4-395E-4C0B-81A7-A37C0C8BB7A7\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"pulmonary_function_test_indicator\": \"YES\", \"initial_pathologic_dx_year\": \"2010\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"day_of_dcc_upload\": \"31\", \"kras_gene_analysis_indicator\": \"NO\", \"submitted_tumor_site\": \"Lung\", \"fev1_percent_ref_postbroncholiator\": \"109\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IB\", \"form_completion_month\": \"7\", \"history_other_malignancy\": \"Yes\", \"tobacco_smoking_year_started\": \"1973\", \"fev1_fvc_ratio_postbroncholiator\": \"101\", \"retrospective_collection\": \"YES\", \"tobacco_smoking_history_indicator\": \"2\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"52\", \"batch_number\": \"144.69.0\", \"disease_code\": \"LUAD\", \"patient_id\": \"6836\", \"year_of_dcc_upload\": \"2016\", \"ajcc_metastasis_pathologic_pm\": \"MX\", \"bcr_patient_barcode\": \"TCGA-91-6836\", \"last_contact_days_to\": \"11\", \"tissue_source_site\": \"91\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Lung\"}, \"id\": \"tcga:0b31dbdf-0623-48e4-a0ec-017650dceda7\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-91-6836\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-19497\", \"death_days_to\": \"1379\", \"tumor_status\": \"TUMOR FREE\", \"residual_tumor\": \"R0\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"anatomic_organ_subdivision\": \"L-Upper\", \"icd_o_3_site\": \"C34.1\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C34.1\", \"vital_status\": \"Dead\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T1\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"0a45f302-5748-48f3-9dc9-66c01843a68e\", \"ajcc_staging_edition\": \"6th\", \"egfr_mutation_status\": \"NO\", \"kras_gene_analysis_indicator\": \"NO\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"initial_pathologic_dx_year\": \"2005\", \"ajcc_nodes_pathologic_pn\": \"N2\", \"day_of_dcc_upload\": \"31\", \"icd_o_3_histology\": \"8140/3\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"submitted_tumor_site\": \"Lung\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIIA\", \"eml4_alk_translocation_status\": \"NO\", \"form_completion_month\": \"7\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"YES\", \"tobacco_smoking_history_indicator\": \"3\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"53\", \"batch_number\": \"160.68.0\", \"disease_code\": \"LUAD\", \"patient_id\": \"6981\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"B0FA6140-7C85-46AA-A5C4-AD724C25077A\", \"bcr_patient_barcode\": \"TCGA-55-6981\", \"form_completion_day\": \"27\", \"tissue_source_site\": \"55\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Lung\"}, \"id\": \"tcga:0a45f302-5748-48f3-9dc9-66c01843a68e\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-55-6981\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-22260\", \"tobacco_smoking_pack_years_smoked\": \"35\", \"tumor_status\": \"TUMOR FREE\", \"residual_tumor\": \"R0\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"YES\", \"anatomic_organ_subdivision\": \"R-Upper\", \"icd_o_3_site\": \"C34.1\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C34.1\", \"vital_status\": \"Alive\", \"egfr_mutation_status\": \"NO\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T1a\", \"form_completion_year\": \"2012\", \"bcr_patient_uuid\": \"afe83af5-af22-49c0-88b4-f1d0dfcbe238\", \"ajcc_staging_edition\": \"7th\", \"fev1_percent_ref_prebroncholiator\": \"79\", \"icd_o_3_histology\": \"8140/3\", \"file_uuid\": \"5E6E2847-FCF3-4225-8DD4-F40653B5222E\", \"treatment_outcome_first_course\": \"Complete Remission/Response\", \"carbon_monoxide_diffusion_dlco\": \"74\", \"pulmonary_function_test_indicator\": \"YES\", \"initial_pathologic_dx_year\": \"2012\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"day_of_dcc_upload\": \"31\", \"kras_gene_analysis_indicator\": \"NO\", \"submitted_tumor_site\": \"Lung\", \"fev1_percent_ref_postbroncholiator\": \"87\", \"initial_pathologic_dx_days_to\": \"0\", \"tobacco_smoking_year_stopped\": \"2008\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IA\", \"eml4_alk_translocation_status\": \"NO\", \"form_completion_month\": \"6\", \"history_other_malignancy\": \"No\", \"tobacco_smoking_year_started\": \"1973\", \"retrospective_collection\": \"NO\", \"tobacco_smoking_history_indicator\": \"4\", \"gender\": \"FEMALE\", \"age_at_initial_pathologic_diagnosis\": \"60\", \"batch_number\": \"213.59.0\", \"disease_code\": \"LUAD\", \"patient_id\": \"8097\", \"year_of_dcc_upload\": \"2016\", \"ajcc_metastasis_pathologic_pm\": \"MX\", \"bcr_patient_barcode\": \"TCGA-55-8097\", \"last_contact_days_to\": \"15\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"form_completion_day\": \"15\", \"tissue_source_site\": \"55\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Lung\"}, \"id\": \"tcga:afe83af5-af22-49c0-88b4-f1d0dfcbe238\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-55-8097\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-24350\", \"tobacco_smoking_pack_years_smoked\": \"58\", \"tumor_status\": \"TUMOR FREE\", \"residual_tumor\": \"R0\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"NO\", \"anatomic_organ_subdivision\": \"R-Upper\", \"disease_code\": \"LUAD\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C34.1\", \"vital_status\": \"Alive\", \"project_code\": \"TCGA\", \"ajcc_tumor_pathologic_pt\": \"T2\", \"form_completion_year\": \"2010\", \"bcr_patient_uuid\": \"e737f650-b72d-44e7-b750-558a56716803\", \"ajcc_staging_edition\": \"6th\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"initial_pathologic_dx_year\": \"2008\", \"ajcc_nodes_pathologic_pn\": \"N1\", \"day_of_dcc_upload\": \"31\", \"icd_o_3_histology\": \"8255/3\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"submitted_tumor_site\": \"Lung\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IIB\", \"form_completion_month\": \"7\", \"history_other_malignancy\": \"No\", \"retrospective_collection\": \"YES\", \"tobacco_smoking_history_indicator\": \"2\", \"gender\": \"MALE\", \"age_at_initial_pathologic_diagnosis\": \"66\", \"batch_number\": \"58.76.0\", \"icd_o_3_site\": \"C34.1\", \"patient_id\": \"4432\", \"year_of_dcc_upload\": \"2016\", \"file_uuid\": \"9EE09A77-F18D-4B83-8ED3-0E959B5E6410\", \"bcr_patient_barcode\": \"TCGA-05-4432\", \"last_contact_days_to\": \"761\", \"form_completion_day\": \"22\", \"tissue_source_site\": \"05\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Lung\"}, \"id\": \"tcga:e737f650-b72d-44e7-b750-558a56716803\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-05-4432\"}\n"}
__init__.py                 42 ERROR    {"path": "/SWIFT/biostream/biostream/tcga/TCGA-LUAD.Individual.json", "error": "pb parse e:Failed to parse attributes field: Message type \"ga4gh.Attributes\" has no field named \"birth_days_to\".\n Available Fields(except extensions): [<google.protobuf.descriptor.FieldDescriptor object at 0x10fad3a10>] cls:Individual {\"attributes\": {\"birth_days_to\": \"-21283\", \"tobacco_smoking_pack_years_smoked\": \"67.5\", \"days_to_last_known_alive\": \"9\", \"pulmonary_function_test_indicator\": \"YES\", \"residual_tumor\": \"RX\", \"month_of_dcc_upload\": \"3\", \"history_neoadjuvant_treatment\": \"No\", \"prospective_collection\": \"YES\", \"anatomic_organ_subdivision\": \"R-Upper\", \"disease_code\": \"LUAD\", \"patient_withdrawal\": \"\\n            \\n        \", \"icd_10\": \"C34.1\", \"form_completion_day\": \"7\", \"file_uuid\": \"AC9D1D72-B83B-46E8-830C-5AB31ED60549\", \"project_code\": \"TCGA\", \"ethnicity\": \"NOT HISPANIC OR LATINO\", \"ajcc_tumor_pathologic_pt\": \"T1a\", \"form_completion_year\": \"2011\", \"bcr_patient_uuid\": \"22302e92-fdfa-462a-9249-efdaa26ee5ed\", \"ajcc_staging_edition\": \"7th\", \"fev1_percent_ref_prebroncholiator\": \"63\", \"performance_status_timing\": \"Preoperative\", \"carbon_monoxide_diffusion_dlco\": \"82\", \"vital_status\": \"Alive\", \"tumor_status\": \"TUMOR FREE\", \"initial_pathologic_dx_year\": \"2011\", \"histologic_diagnosis\": \"Lung Adenocarcinoma\", \"day_of_dcc_upload\": \"31\", \"ecog_score\": \"0\", \"icd_o_3_histology\": \"8140/3\", \"submitted_tumor_site\": \"Lung\", \"fev1_percent_ref_postbroncholiator\": \"77\", \"initial_pathologic_dx_days_to\": \"0\", \"patient_consent_status\": \"YES\", \"ajcc_pathologic_tumor_stage\": \"Stage IA\", \"form_completion_month\": \"6\", \"history_other_malignancy\": \"No\", \"tobacco_smoking_year_started\": \"1968\", \"fev1_fvc_ratio_postbroncholiator\": \"97\", \"retrospective_collection\": \"NO\", \"tobacco_smoking_history_indicator\": \"2\", \"gender\": \"MALE\", \"age_at_initial_pathologic_diagnosis\": \"58\", \"fev1_fvc_ratio_prebroncholiator\": \"87\", \"batch_number\": \"119.75.0\", \"icd_o_3_site\": \"C34.1\", \"patient_id\": \"6144\", \"year_of_dcc_upload\": \"2016\", \"ajcc_metastasis_pathologic_pm\": \"M0\", \"bcr_patient_barcode\": \"TCGA-44-6144\", \"last_contact_days_to\": \"9\", \"ajcc_nodes_pathologic_pn\": \"N0\", \"tissue_source_site\": \"44\", \"bcr\": \"Nationwide Children's Hospital\", \"tumorSite\": \"Lung\"}, \"id\": \"tcga:22302e92-fdfa-462a-9249-efdaa26ee5ed\", \"datasetId\": \"TCGA-LUAD\", \"name\": \"TCGA-44-6144\"}\n"}
===================== 4 failed, 2 passed in 14.96 seconds ======================

```
